<!DOCTYPE html> <html lang="en-us"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Publications | BCEM </title> <meta name="author" content="BCEM @ Uniandes"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/bcem_logo.png?526fa6badc25bb3e530d9bf4a500aee4"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://alejandrocs98.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">BCEM</span> @ Uniandes </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/team/">Team </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">Research </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">Repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/contact/">Contact </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Pics </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/generations/">BCEM along the years</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/gallery/">Gallery</a> </div> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> EN <span class="fi fi-us"></span> </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/pt-br/publications/"> PT <span class="fi fi-br"></span> </a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link"> <i class="ti ti-search"></i></span> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div id="turner_abolishment_2023" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1007/s00705-022-05694-2" rel="external nofollow noopener">Abolishment of morphology-based taxa and change to binomial species names: 2022 taxonomy update of the ICTV bacterial viruses subcommittee</a></div> <div class="author"> Dann Turner, Andrey N. Shkoporov, Cédric Lood, and <span class="more-authors" title="click to view 33 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '33 more authors' ? 'Andrew D. Millard, Bas E. Dutilh, Poliane Alfenas-Zerbini, Leonardo J. Zyl, Ramy K. Aziz, Hanna M. Oksanen, Minna M. Poranen, Andrew M. Kropinski, Jakub Barylski, J. Rodney Brister, Nina Chanisvili, Rob A. Edwards, François Enault, Annika Gillis, Petar Knezevic, Mart Krupovic, Ipek Kurtböke, Alla Kushkina, Rob Lavigne, Susan Lehman, Malgorzata Lobocka, Cristina Moraru, Andrea Moreno Switt, Vera Morozova, Jesca Nakavuma, Alejandro Reyes Muñoz, Jānis Rūmnieks, BL Sarkar, Matthew B. Sullivan, Jumpei Uchiyama, Johannes Wittmann, Tong Yigang, Evelien M. Adriaenssens' : '33 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">33 more authors</span> </div> <div class="periodical"> <em>Archives of Virology</em>, Jan 2023 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1007/s00705-022-05694-2" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>This article summarises the activities of the Bacterial Viruses Subcommittee of the International Committee on Taxonomy of Viruses for the period of March 2021−March 2022. We provide an overview of the new taxa proposed in 2021, approved by the Executive Committee, and ratified by vote in 2022. Significant changes to the taxonomy of bacterial viruses were introduced: the paraphyletic morphological families Podoviridae, Siphoviridae, and Myoviridae as well as the order Caudovirales were abolished, and a binomial system of nomenclature for species was established. In addition, one order, 22 families, 30 subfamilies, 321 genera, and 862 species were newly created, promoted, or moved.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="simmonds_four_2023" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1371/journal.pbio.3001922" rel="external nofollow noopener">Four principles to establish a universal virus taxonomy</a></div> <div class="author"> Peter Simmonds, Evelien M. Adriaenssens, F. Murilo Zerbini, and <span class="more-authors" title="click to view 40 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '40 more authors' ? 'Nicola G. A. Abrescia, Pakorn Aiewsakun, Poliane Alfenas-Zerbini, Yiming Bao, Jakub Barylski, Christian Drosten, Siobain Duffy, W. Paul Duprex, Bas E. Dutilh, Santiago F. Elena, Maria Laura García, Sandra Junglen, Aris Katzourakis, Eugene V. Koonin, Mart Krupovic, Jens H. Kuhn, Amy J. Lambert, Elliot J. Lefkowitz, Małgorzata Łobocka, Cédric Lood, Jennifer Mahony, Jan P. Meier-Kolthoff, Arcady R. Mushegian, Hanna M. Oksanen, Minna M. Poranen, Alejandro Reyes-Muñoz, David L. Robertson, Simon Roux, Luisa Rubino, Sead Sabanadzovic, Stuart Siddell, Tim Skern, Donald B. Smith, Matthew B. Sullivan, Nobuhiro Suzuki, Dann Turner, Koenraad Van Doorslaer, Anne-Mieke Vandamme, Arvind Varsani, Nikos Vasilakis' : '40 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">40 more authors</span> </div> <div class="periodical"> <em>PLOS Biology</em>, Feb 2023 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001922" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>A universal taxonomy of viruses is essential for a comprehensive view of the virus world and for communicating the complicated evolutionary relationships among viruses. However, there are major differences in the conceptualisation and approaches to virus classification and nomenclature among virologists, clinicians, agronomists, and other interested parties. Here, we provide recommendations to guide the construction of a coherent and comprehensive virus taxonomy, based on expert scientific consensus. Firstly, assignments of viruses should be congruent with the best attainable reconstruction of their evolutionary histories, i.e., taxa should be monophyletic. This fundamental principle for classification of viruses is currently included in the International Committee on Taxonomy of Viruses (ICTV) code only for the rank of species. Secondly, phenotypic and ecological properties of viruses may inform, but not override, evolutionary relatedness in the placement of ranks. Thirdly, alternative classifications that consider phenotypic attributes, such as being vector-borne (e.g., “arboviruses”), infecting a certain type of host (e.g., “mycoviruses,” “bacteriophages”) or displaying specific pathogenicity (e.g., “human immunodeficiency viruses”), may serve important clinical and regulatory purposes but often create polyphyletic categories that do not reflect evolutionary relationships. Nevertheless, such classifications ought to be maintained if they serve the needs of specific communities or play a practical clinical or regulatory role. However, they should not be considered or called taxonomies. Finally, while an evolution-based framework enables viruses discovered by metagenomics to be incorporated into the ICTV taxonomy, there are essential requirements for quality control of the sequence data used for these assignments. Combined, these four principles will enable future development and expansion of virus taxonomy as the true evolutionary diversity of viruses becomes apparent.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="oliveira_rational_2023" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3390/v15020519" rel="external nofollow noopener">Rational Design of Profile HMMs for Sensitive and Specific Sequence Detection with Case Studies Applied to Viruses, Bacteriophages, and Casposons</a></div> <div class="author"> Liliane S. Oliveira, Alejandro Reyes, Bas E. Dutilh, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Arthur Gruber' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Viruses</em>, Feb 2023 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3390/v15020519" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Profile hidden Markov models (HMMs) are a powerful way of modeling biological sequence diversity and constitute a very sensitive approach to detecting divergent sequences. Here, we report the development of protocols for the rational design of profile HMMs. These methods were implemented on TABAJARA, a program that can be used to either detect all biological sequences of a group or discriminate specific groups of sequences. By calculating position-specific information scores along a multiple sequence alignment, TABAJARA automatically identifies the most informative sequence motifs and uses them to construct profile HMMs. As a proof-of-principle, we applied TABAJARA to generate profile HMMs for the detection and classification of two viral groups presenting different evolutionary rates: bacteriophages of the Microviridae family and viruses of the Flavivirus genus. We obtained conserved models for the generic detection of any Microviridae or Flavivirus sequence, and profile HMMs that can specifically discriminate Microviridae subfamilies or Flavivirus species. In another application, we constructed Cas1 endonuclease-derived profile HMMs that can discriminate CRISPRs and casposons, two evolutionarily related transposable elements. We believe that the protocols described here, and implemented on TABAJARA, constitute a generic toolbox for generating profile HMMs for the highly sensitive and specific detection of sequence classes.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="sutcliffe_bacteriophages_2023" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1016/j.isci.2023.106007" rel="external nofollow noopener">Bacteriophages playing nice: Lysogenic bacteriophage replication stable in the human gut microbiota</a></div> <div class="author"> Steven G. Sutcliffe, Alejandro Reyes, and Corinne F. Maurice </div> <div class="periodical"> <em>iScience</em>, Feb 2023 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1016/j.isci.2023.106007" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Bacteriophages, viruses specific to bacteria, coexist with their bacterial hosts with limited diversity fluctuations in the guts of healthy individuals where they replicate mostly via lysogenic replication. This favors ‘piggy-back-the-winner’ over ‘kill-the-winner’ dynamics which are driven by lytic bacteriophage replication. Revisiting the deep-viral sequencing data of a healthy individual sampled over 2.4 years, we explore how these dynamics occur. Prophages found in assembled bacterial metagenomes were also found extra-cellularly, as induced phage particles (iPPs), likely derived from prophage activation. These iPPs were diverse and continually present in low abundance, relative to the highly abundant but less diverse lytic phage population. The continuous detection of low levels of iPPs suggests that spontaneous induction regularly occurs in this healthy individual, possibly allowing prophages to maintain their ability to replicate and avoiding degradation and loss from the gut microbiota.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="castellanos_genomic_2023" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1128/spectrum.04851-22" rel="external nofollow noopener">Genomic and Evolutionary Features of Nine AHPND Positive Vibrio parahaemolyticus Strains Isolated from South American Shrimp Farms</a></div> <div class="author"> Alejandro Castellanos, Leda Restrepo, Leandro Bajaña, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Irma Betancourt, Bonny Bayot, Alejandro Reyes' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Microbiology Spectrum</em>, Jun 2023 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1128/spectrum.04851-22" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Vibrio parahaemolyticus is a bacterial pathogen that becomes lethal to Penaeus shrimps when acquiring the pVA1-type plasmid carrying the PirABvp genes, causing acute hepatopancreatic necrosis disease (AHPND). This disease causes significant losses across the world, with outbreaks reported in Southeast Asia, Mexico, and South America. Virulence level and mortality differences have been reported in isolates from different locations, and whether this phenomenon is caused by plasmid-related elements or genomic-related elements from the bacteria remains unclear. Here, nine genomes of South American AHPND-causing V. parahaemolyticus (VPAHPND) isolates were assembled and analyzed using a comparative genomics approach at (i) whole-genome, (ii) secretion system, and (iii) plasmid level, and then included for a phylogenomic analysis with another 86 strains. Two main results were obtained from our analyses. First, all isolates contained pVA1-type plasmids harboring the toxin coding genes, and with high similarity with the prototypical sequence of Mexican-like origin, while phylogenomic analysis showed some level of heterogeneity with discrete clusters and wide diversity compared to other available genomes. Second, although a high genomic similarity was observed, variation in virulence genes and clusters was observed, which might be relevant in the expression of the disease. Overall, our results suggest that South American pathogenic isolates are derived from various genetic lineages which appear to have acquired the plasmid through horizontal gene transfer. Furthermore, pathogenicity seems to be a multifactorial trait where the degree of virulence could be altered by the presence or variations of several virulence factors.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="perez-quintero_comparative_2023" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1094/PHYTO-12-22-0477-SC" rel="external nofollow noopener">Comparative Genomics Identifies Conserved and Variable TAL Effectors in African Strains of the Cotton Pathogen Xanthomonas citri pv. malvacearum</a></div> <div class="author"> Alvaro L. Pérez-Quintero, Luis M. Rodriguez-R, Sara Cuesta-Morrondo, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Eliška Hakalová, Daniela Betancurt-Anzola, Laura Carolina Camelo Valera, Luis Alberto Chica Cardenas, Luisa Matiz-Céron, Jonathan M. Jacobs, Veronica Roman-Reyna, Alejandro Reyes Muñoz, Adriana Jimena Bernal Giraldo, Ralf Koebnik' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>Phytopathology®</em>, Aug 2023 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1094/PHYTO-12-22-0477-SC" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Strains of Xanthomonas citri pv. malvacearum cause bacterial blight of cotton, a potentially serious threat to cotton production worldwide, including in sub-Saharan countries. Development of disease symptoms, such as water soaking, has been linked to the activity of a class of type 3 effectors, called transcription activator-like (TAL) effectors, which induce susceptibility genes in the host’s cells. To gain further insight into the global diversity of the pathogen, to elucidate their repertoires of TAL effector genes, and to better understand the evolution of these genes in the cotton-pathogenic xanthomonads, we sequenced the genomes of three African strains of X. citri pv. malvacearum using nanopore technology. We show that the cotton-pathogenic pathovar of X. citri is a monophyletic lineage containing at least three distinct genetic subclades, which appear to be mirrored by their repertoires of TAL effectors. We observed an atypical level of TAL effector gene pseudogenization, which might be related to resistance genes that are deployed to control the disease. Our work thus contributes to a better understanding of the conservation and importance of TAL effectors in the interaction with the host plant, which can inform strategies for improving resistance against bacterial blight in cotton.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="rangel-pineros_virify_2023" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1371/journal.pcbi.1011422" rel="external nofollow noopener">VIRify: An integrated detection, annotation and taxonomic classification pipeline using virus-specific protein profile hidden Markov models</a></div> <div class="author"> Guillermo Rangel-Pineros, Alexandre Almeida, Martin Beracochea, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Ekaterina Sakharova, Manja Marz, Alejandro Reyes Muñoz, Martin Hölzer, Robert D. Finn' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>PLOS Computational Biology</em>, Aug 2023 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1371/journal.pcbi.1011422" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The study of viral communities has revealed the enormous diversity and impact these biological entities have on various ecosystems. These observations have sparked widespread interest in developing computational strategies that support the comprehensive characterisation of viral communities based on sequencing data. Here we introduce VIRify, a new computational pipeline designed to provide a user-friendly and accurate functional and taxonomic characterisation of viral communities. VIRify identifies viral contigs and prophages from metagenomic assemblies and annotates them using a collection of viral profile hidden Markov models (HMMs). These include our manually-curated profile HMMs, which serve as specific taxonomic markers for a wide range of prokaryotic and eukaryotic viral taxa and are thus used to reliably classify viral contigs. We tested VIRify on assemblies from two microbial mock communities, a large metagenomics study, and a collection of publicly available viral genomic sequences from the human gut. The results showed that VIRify could identify sequences from both prokaryotic and eukaryotic viruses, and provided taxonomic classifications from the genus to the family rank with an average accuracy of 86.6%. In addition, VIRify allowed the detection and taxonomic classification of a range of prokaryotic and eukaryotic viruses present in 243 marine metagenomic assemblies. Finally, the use of VIRify led to a large expansion in the number of taxonomically classified human gut viral sequences and the improvement of outdated and shallow taxonomic classifications. Overall, we demonstrate that VIRify is a novel and powerful resource that offers an enhanced capability to detect a broad range of viral contigs and taxonomically classify them.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="adriaenssens_guidelines_2023" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/s41587-023-01844-2" rel="external nofollow noopener">Guidelines for public database submission of uncultivated virus genome sequences for taxonomic classification</a></div> <div class="author"> Evelien M. Adriaenssens, Simon Roux, J. Rodney Brister, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Ilene Karsch-Mizrachi, Jens H. Kuhn, Arvind Varsani, Tong Yigang, Alejandro Reyes, Cédric Lood, Elliot J. Lefkowitz, Matthew B. Sullivan, Robert A. Edwards, Peter Simmonds, Luisa Rubino, Sead Sabanadzovic, Mart Krupovic, Bas E. Dutilh' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>Nature Biotechnology</em>, Jul 2023 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/s41587-023-01844-2" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div id="villabona_conserved_2023" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1128/msphere.00139-23" rel="external nofollow noopener">Conserved, yet disruption-prone, gut microbiomes in neotropical bumblebees</a></div> <div class="author"> Nickole Villabona, Nancy Moran, Tobin Hammer, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Alejandro Reyes' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>mSphere</em>, Oct 2023 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1128/msphere.00139-23" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Bumblebees are important pollinators in natural ecosystems and agriculture, but many species are declining. Temperate-zone bumblebees have host-specific and beneficial gut microbiomes, which may have a role in mediating the effects of stressors. However, there is almost no published information on the gut microbiomes of tropical bumblebees. As temperate and tropical bumblebees encounter different floral resources and environmental conditions, their microbiomes could differ. Here, we characterized the gut microbiomes of four neotropical Bombus species and, for comparison, co-occurring solitary bees (genus Thygater). We collected wild-foraging bees from multiple sites in central Colombia and used 16S rRNA gene sequencing to characterize their gut microbiomes. DNA barcoding and morphology were used to identify bumblebee species. We found that the microbiomes of neotropical bumblebees cluster with those of closely related temperate-zone species, in agreement with a model of bumblebee-symbiont codiversification. There was no evidence of geographic differences in microbiome composition between neotropical and temperate-zone bumblebees. These results suggest that the microbiome was conserved during bumblebee dispersal from North America, despite major shifts in ecology and life history. As previously observed in temperate-zone species, some neotropical bumblebees have highly disrupted microbiomes, in which conserved gut bacterial symbionts are replaced by environmental microbes. In these individuals, the gut microbial profile is more like that of solitary bees than of conspecifics. The gut parasites Nosema and Crithidia are also prevalent and associated with microbiome disruption. Our findings provide insights into the biogeography of bee microbiomes and a foundation for studying bee-microbe-stressor interactions in the neotropics.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div id="pena_gonzalez_analysis_2022" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.7490/f1000research.1118938.1" rel="external nofollow noopener">Analysis and exploration of microbial traits in a wet coffee fermentation experiment using MGnify</a></div> <div class="author"> Angela Peña González, and Alejandro Reyes Muñoz </div> <div class="periodical"> <em>F1000Research</em>, Apr 2022 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.7490/f1000research.1118938.1" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>This CABANA e-Learning tutorial will guide you through the use of the EMBL-EBI web server MGnify to process and analyze multiple shotgun metagenomic datasets derived from a wet coffee fermentation process at different time points.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="matiz-ceron_taxonomical_2022" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fpls.2021.782663" rel="external nofollow noopener">Taxonomical Evaluation of Plant Chloroplastic Markers by Bayesian Classifier</a></div> <div class="author"> Luisa Matiz-Ceron, Alejandro Reyes, and Juan Anzola </div> <div class="periodical"> <em>Frontiers in Plant Science</em>, Apr 2022 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fpls.2021.782663" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>DNA barcodes are standardized sequences that range between 400 and 800 bp, vary at different taxonomic levels, and make it possible to assign sequences to species that have been previously taxonomically characterized. Several DNA barcodes have been postulated for plants, nonetheless, their classification potential has not been evaluated for metabarcoding, and as a result, it would appear as none of them excels above the others in this area. One tool that has been widely used and served as a baseline when evaluating new approaches is Naïve Bayesian Classifiers (NBC). The present study aims at evaluating the classification power of several plant chloroplast genetic markers that have been proposed as barcodes (trnL, rpoB, rbcL, matK, psbA-trnH, and psbK) using an NBC. We performed the classification at different taxonomic levels, and identified problematic genera when resolution was desired. We propose matK and trnL as potential candidate markers with resolution up to genus level. Some problematic genera within certain families could lead to the misclassification no matter which marker is used (i.e., Aegilops, Gueldenstaedtia, Helianthus, Oryza, Shorea, Thysananthus, and Triticum). Finally, we suggest recommendations for the taxonomic identification of plants in samples with potential mixtures.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="hernandez_functional_2022" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fmicb.2022.813002" rel="external nofollow noopener">Functional and Phylogenetic Characterization of Bacteria in Bovine Rumen Using Fractionation of Ruminal Fluid</a></div> <div class="author"> Ruth Hernández, Maryam Chaib De Mares, Hugo Jimenez, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Alejandro Reyes, Alejandro Caro-Quintero' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Apr 2022 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2022.813002" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Cattle productivity depends on our ability to fully understand and manipulate the fermentation process of plant material that occurs in the bovine rumen, which ultimately leads to the improvement of animal health and increased productivity with a reduction in environmental impact. An essential step in this direction is the phylogenetic and functional characterization of the microbial species composing the ruminal microbiota. To address this challenge, we separated a ruminal fluid sample by size and density using a sucrose density gradient. We used the full sample and the smallest fraction (5%), allowing the enrichment of bacteria, to assemble metagenome-assembled genomes (MAGs). We obtained a total of 16 bacterial genomes, 15 of these enriched in the smallest fraction of the gradient. According to the recently proposed Genome Taxonomy Database (GTDB) taxonomy, these MAGs belong to Bacteroidota, Firmicutes_A, Firmicutes, Proteobacteria, and Spirochaetota phyla. Fifteen MAGs were novel at the species level and four at the genus level. The functional characterization of these MAGs suggests differences from what is currently known from the genomic potential of well-characterized members from this complex environment. Species of the phyla Bacteroidota and Spirochaetota show the potential for hydrolysis of complex polysaccharides in the plant cell wall and toward the production of B-complex vitamins and protein degradation in the rumen. Conversely, the MAGs belonging to Firmicutes and Alphaproteobacteria showed a reduction in several metabolic pathways; however, they have genes for lactate fermentation and the presence of hydrolases and esterases related to chitin degradation. Our results demonstrate that the separation of the rumen microbial community by size and density reduced the complexity of the ruminal fluid sample and enriched some poorly characterized ruminal bacteria allowing exploration of their genomic potential and their functional role in the rumen ecosystem.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="varunjikar_comparing_2022" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1016/j.foodcont.2021.108417" rel="external nofollow noopener">Comparing novel shotgun DNA sequencing and state-of-the-art proteomics approaches for authentication of fish species in mixed samples</a></div> <div class="author"> Madhushri S. Varunjikar, Carlos Moreno-Ibarguen, Juan S. Andrade-Martinez, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Hui-Shan Tung, Ikram Belghit, Magnus Palmblad, Pål A. Olsvik, Alejandro Reyes, Josef D. Rasinger, Kai K. Lie' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Food Control</em>, Jan 2022 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1016/j.foodcont.2021.108417" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Replacement of high-value fish species with cheaper varieties or mislabelling of food unfit for human consumption is a global problem violating both consumers’ rights and safety. For distinguishing fish species in pure samples, DNA approaches are available; however, authentication and quantification of fish species in mixtures remains a challenge. In the present study, a novel high-throughput shotgun DNA sequencing approach applying masked reference libraries was developed and used for authentication and abundance calculations of fish species in mixed samples. Results demonstrate that the analytical protocol presented here can discriminate and predict relative abundances of different fish species in mixed samples with high accuracy. In addition to DNA analyses, shotgun proteomics tools based on direct spectra comparisons were employed on the same mixture. Similar to the DNA approach, the identification of individual fish species and the estimation of their respective relative abundances in a mixed sample also were feasible. Furthermore, the data obtained indicated that DNA sequencing using masked libraries predicted species-composition of the fish mixture with higher specificity, while at a taxonomic family level, relative abundances of the different species in the fish mixture were predicted with slightly higher accuracy using proteomics tools. Taken together, the results demonstrate that both DNA and protein-based approaches presented here can be used to efficiently tackle current challenges in feed and food authentication analyses.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="adriaenssens_taxonomy_2022" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/" rel="external nofollow noopener">Taxonomy proposal 2021: Abolish the order Caudovirales and the families Myoviridae, Siphoviridae and Podoviridae (Caudoviricetes)</a></div> <div class="author"> Evelien Adriaenssens, Andrew M. Kropinski, Turner D, and <span class="more-authors" title="click to view 28 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '28 more authors' ? 'Mart Krupovic, Andrew Millard, Bas E. Dutilh, Hanna M Oksanen, Cedric Lood, Alejandro Reyes, Cristina Moraru, Ramy K Aziz, Tong Yigang, Jakub Barylski, Malgorzata Lobocka, B L Sarkar, Annika Gillis, Leonardo J. Zyl, Susan M Lehman, Matthew B. Sullivan, Johannes Wittmann, Alla I Kushkina, Petar Knezevic, J. Rodney Brister, Rob A Edwards, Ipek Kurtboke, Minna Poranen, Rob Lavigne, Andrey N Shkoporov, Jumpei Uchiyama, Andrea I. Moreno Switt, François Enault' : '28 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">28 more authors</span> </div> <div class="periodical"> <em>In ICTV Online: International Committee on Taxonomy of Viruses (ICTV)</em> , Jan 2022 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div id="clavijo_gut_2022" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/s41598-021-04679-6" rel="external nofollow noopener">The gut microbiota of chickens in a commercial farm treated with a Salmonella phage cocktail</a></div> <div class="author"> Viviana Clavijo, Tatiana Morales, Martha Josefina Vives-Flores, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Alejandro Reyes Muñoz' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Scientific Reports</em>, Jan 2022 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/s41598-021-04679-6" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The microbiota in broiler chicken intestines affects the animals’ health, metabolism, and immunity both positively and negatively. Accordingly, it has a significant impact on animal productivity. Phages, host-specific parasites of bacterial cells, are a promising antimicrobial alternative that selectively target pathogens without disturbing the microbiota. The purpose of this study is to further characterize the commensal microbial community at production scale in broiler chickens treated with a Salmonella phage treatment. We evaluated the cecal microbiota of broilers reared in a commercial farming system where a phage cocktail against Salmonella, SalmoFree was supplied to animals. To do so, two field trials were conducted, incorporating three doses of phages in the broilers’ drinking water. Our results showed that the core microbiome (taxa that were present in more than 50% of samples) contained species that are key to microbiota adaptation in the last stage of the production cycle. Among these, there are some important degraders of complex polysaccharides and producers of short chain fatty acids (SCFA) such as Eisenbergiella and Lachnoclostridium. The phage cocktail did not affect the normal development of the microbiota’s structure. The addition of the phage cocktail resulted in a significant reduction in Campylobacter and an increase in Butyricimonas, Helicobacter and Rikenellaceae, which are common inhabitants in chicken gut with known negative and positive effects on their health and metabolism. Altogether, we consider that these results contribute valuable information to the implementation of large-scale phage therapy technologies.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="andrade-martinez_computational_2022" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1128/mmbr.00004-21" rel="external nofollow noopener">Computational Tools for the Analysis of Uncultivated Phage Genomes</a></div> <div class="author"> Juan Sebastián Andrade-Martínez, Laura Carolina Camelo Valera, Luis Alberto Chica Cárdenas, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Laura Forero-Junco, Gamaliel López-Leal, J. Leonardo Moreno-Gallego, Guillermo Rangel-Pineros, Alejandro Reyes' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Microbiology and Molecular Biology Reviews</em>, Mar 2022 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1128/mmbr.00004-21" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>SUMMARY Over a century of bacteriophage research has uncovered a plethora of fundamental aspects of their biology, ecology, and evolution. Furthermore, the introduction of community-level studies through metagenomics has revealed unprecedented insights on the impact that phages have on a range of ecological and physiological processes. It was not until the introduction of viral metagenomics that we began to grasp the astonishing breadth of genetic diversity encompassed by phage genomes. Novel phage genomes have been reported from a diverse range of biomes at an increasing rate, which has prompted the development of computational tools that support the multilevel characterization of these novel phages based solely on their genome sequences. The impact of these technologies has been so large that, together with MAGs (Metagenomic Assembled Genomes), we now have UViGs (Uncultivated Viral Genomes), which are now officially recognized by the International Committee for the Taxonomy of Viruses (ICTV), and new taxonomic groups can now be created based exclusively on genomic sequence information. Even though the available tools have immensely contributed to our knowledge of phage diversity and ecology, the ongoing surge in software programs makes it challenging to keep up with them and the purpose each one is designed for. Therefore, in this review, we describe a comprehensive set of currently available computational tools designed for the characterization of phage genome sequences, focusing on five specific analyses: (i) assembly and identification of phage and prophage sequences, (ii) phage genome annotation, (iii) phage taxonomic classification, (iv) phage-host interaction analysis, and (v) phage microdiversity.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="lopez-leal_mining_2022" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1128/msystems.00326-22" rel="external nofollow noopener">Mining of Thousands of Prokaryotic Genomes Reveals High Abundance of Prophages with a Strictly Narrow Host Range</a></div> <div class="author"> Gamaliel López-Leal, Laura Carolina Camelo-Valera, Juan Manuel Hurtado-Ramírez, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Jérôme Verleyen, Santiago Castillo-Ramírez, Alejandro Reyes-Muñoz' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>mSystems</em>, Jul 2022 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1128/msystems.00326-22" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Phages and prophages are one of the principal modulators of microbial populations. However, much of their diversity is still poorly understood. Here, we extracted 33,624 prophages from 13,713 complete prokaryotic genomes to explore the prophage diversity and their relationships with their host. Our results reveal that prophages were present in 75% of the genomes studied. In addition, Enterobacterales were significantly enriched in prophages. We also found that pathogens are a significant reservoir of prophages. Finally, we determined that the prophage relatedness and the range of genomic hosts were delimited by the evolutionary relationships of their hosts. On a broader level, we got insights into the prophage population, identified in thousands of publicly available prokaryotic genomes, by comparing the prophage distribution and relatedness between them and their hosts. IMPORTANCE Phages and prophages play an essential role in controlling their host populations either by modulating the host abundance or providing them with genes that benefit the host. The constant growth in next-generation sequencing technology has caused the development of powerful computational tools to identify phages and prophages with high precision. Making it possible to explore the prophage populations integrated into host genomes on a large scale. However, it is still a new and under-explored area, and efforts are still required to identify prophage populations to understand their dynamics with their hosts.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div id="pena_gonzalez_practical_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.7490/f1000research.1118763.1" rel="external nofollow noopener">Practical metagenomics: The Study of human gut microbiome in health and disease: Applications in Acute Diarrheal Diseases (ADD)</a></div> <div class="author"> Angela Peña González, Alejandro Reyes Muñoz, and Alejandro Madrigal Leiva </div> <div class="periodical"> <em>F1000Research</em>, Aug 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.7490/f1000research.1118763.1" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>This CABANA e-Learning tutorial will guide you through the analysis of shotgun metagenomic datasets derived from clinical samples (stool) in a practical way using command line and graphical user interfaces (GUIs). You might want to use this workflow as a guide through the different analyses. Keep in mind that since this tutorial is mainly designed for educational purposes, we performed an initial data subsampling step. However, for regular studies we encourage the use of complete datasets.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="hernandez_disentangling_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fmicb.2021.664754" rel="external nofollow noopener">Disentangling the Complexity of the Rumen Microbial Diversity Through Fractionation Using a Sucrose Density Gradient</a></div> <div class="author"> Ruth Hernández, Hugo Jimenez, Cesar Vargas-Garcia, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Alejandro Caro-Quintero, Alejandro Reyes' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Aug 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2021.664754" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The ruminal microbial community is an important element in health, nutrition, livestock productivity, and climate impact. Despite the historic and current efforts to characterize this microbial diversity, many of its members remain unidentified, making it challenging to associate microbial groups with functions. Here we present a low-cost methodology for rumen sample treatment that separates the microbial community based on cell size, allowing for the identification of subtle compositional changes. In brief, the sample is centrifuged through a series of sucrose density gradients, and cells migrate to their corresponding density fraction. From each fraction, DNA is extracted and 16S rRNA gene amplicons are sequenced. We tested our methodology on four animals under two different conditions, fasting, and post-feeding. Each fraction was examined by confocal microscopy showing that the same sucrose fraction consistently separated similar cell-sized microorganisms independent of the animal or treatment. Microbial composition analysis using metabarcoding showed that our methodology detected low abundance bacterial families and population changes between fasting and post-feeding treatments that could not be observed by bulk DNA analysis. In conclusion, the sucrose-based method is a powerful low-cost approximation to untwine, enrich, and potentially isolate uncharacterized members of the ruminal microbiome.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="ras_challenges_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/feduc.2021.710971" rel="external nofollow noopener">Challenges and Considerations for Delivering Bioinformatics Training in LMICs: Perspectives From Pan-African and Latin American Bioinformatics Networks</a></div> <div class="author"> Verena Ras, Patricia Carvajal-López, Piraveen Gopalasingam, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Alice Matimba, Paballo Abel Chauke, Nicola Mulder, Fatma Guerfali, Victoria Dominguez Del Angel, Alejandro Reyes, Guilherme Oliveira, Javier De Las Rivas, Marco Cristancho' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Frontiers in Education</em>, Aug 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/feduc.2021.710971" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div id="vasquez_non-extensive_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fchem.2021.700802" rel="external nofollow noopener">Non-Extensive Fragmentation of Natural Products and Pharmacophore-Based Virtual Screening as a Practical Approach to Identify Novel Promising Chemical Scaffolds</a></div> <div class="author"> Andrés Felipe Vásquez, Alejandro Reyes Muñoz, Jorge Duitama, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Andrés González Barrios' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Frontiers in Chemistry</em>, Aug 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fchem.2021.700802" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Fragment-based drug design (FBDD) and pharmacophore modeling have proven to be efficient tools to discover novel drugs. However, these approaches may become limited if the collection of fragments is highly repetitive, poorly diverse, or excessively simple. In this article, combining pharmacophore modeling and a non-classical type of fragmentation (herein called non-extensive) to screen a natural product (NP) library may provide fragments predicted as potent, diverse, and developable. Initially, we applied retrosynthetic combinatorial analysis procedure (RECAP) rules in two versions, extensive and non-extensive, in order to deconstruct a virtual library of NPs formed by the databases Traditional Chinese Medicine (TCM), AfroDb (African Medicinal Plants database), NuBBE (Nuclei of Bioassays, Biosynthesis, and Ecophysiology of Natural Products), and UEFS (Universidade Estadual de Feira de Santana). We then developed a virtual screening (VS) using two groups of natural-product-derived fragments (extensive and non-extensive NPDFs) and two overlapping pharmacophore models for each of 20 different proteins of therapeutic interest. Molecular weight, lipophilicity, and molecular complexity were estimated and compared for both types of NPDFs (and their original NPs) before and after the VS proceedings. As a result, we found that non-extensive NPDFs exhibited a much higher number of chemical entities compared to extensive NPDFs (45,355 vs. 11,525 compounds), accounting for the larger part of the hits recovered and being far less repetitive than extensive NPDFs. The structural diversity of both types of NPDFs and the NPs was shown to diminish slightly after VS procedures. Finally, and most interestingly, the pharmacophore fit score of the non-extensive NPDFs proved to be not only higher, on average, than extensive NPDFs (56% of cases) but also higher than their original NPs (69% of cases) when all of them were also recognized as hits after the VS. The findings obtained in this study indicated that the proposed cascade approach was useful to enhance the probability of identifying innovative chemical scaffolds, which deserve further development to become drug-sized candidate compounds. We consider that the knowledge about the deconstruction degree required to produce NPDFs of interest represents a good starting point for eventual synthesis, characterization, and biological activity studies.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="adriaenssens_abolish_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/" rel="external nofollow noopener">Abolish the order Caudovirales and the families Myoviridae, Siphoviridae and Podoviridae (Caudoviricetes)</a></div> <div class="author"> Evelien Adriaenssens, AM Kropinski, Dan Turner, and <span class="more-authors" title="click to view 28 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '28 more authors' ? 'Mart Krupovic, A Millard, BE Dutilh, HM Oksanen, C Lood, A Reyes, C Moraru, RK Aziz, Y Tong, J Barylski, M Łobocka, BL Sarkar, A Gillis, LJ Zyl, SM Lehman, M Sullivan, J Wittman, AI Kushkina, P Knezevic, JR Brister, RA Edwards, DI Kurtböke, MM Poranen, R Lavigne, AN Shkoporov, J Uchiyama, AI Moreno-Switt, F Enault' : '28 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">28 more authors</span> </div> <div class="periodical"> <em>In International Committee on Taxonomy of Viruses</em> , Aug 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Here we propose to abolish the order Caudovirales keeping all underlying taxonomic ranks in the class Caudoviricetes. We also propose to abolish the morphology-based phage families Myoviridae, Siphoviridae and Podoviridae which have been repeatedly shown not to be monophyletic.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="cardenas_bacterial_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.7717/peerj.10571" rel="external nofollow noopener">Bacterial meta-analysis of chicken cecal microbiota</a></div> <div class="author"> Luis Alberto Chica Cardenas, Viviana Clavijo, Martha Vives, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Alejandro Reyes' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>PeerJ</em>, Jan 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.7717/peerj.10571" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Poultry production is an industry that generates 90,000 metric tons of chicken meat worldwide. Thus, optimizing chicken growth and sustainable production is of great importance. A central factor determining not only production parameters, but also stability of the immune system and chicken health, is the diversity and variability of the microbiota present throughout the gastrointestinal tract. To date, several studies have investigated the relationship between bacterial communities and the gut microbiome, with limited data to compare. This study aims to create a bacterial meta-analysis based on studies using amplicon sequencing with Illumina sequencing technologies in order to build a baseline for comparison in future analyses of the cecal bacterial composition in chicken. A systematic literature review was performed (SYRF ID: e84f0468-e418-4eec-9da4-b517f1b4809d. Full project URL: https://app.syrf.org.uk/projects/e84f0468-e418-4eec-9da4-b517f1b4809d/detail). From all the available and analyzed manuscripts only nine contained full raw-sequence data available and the corresponding metadata. A total of 324 samples, comprising three different regions within the 16S rRNA gene, were analyzed. Due to the heterogeneity of the data, each region was analyzed independently and an effort for a joint analysis was performed as well. Taxonomic profiling revealed 11 phyla, with Firmicutes as the most prevalent phylum, followed by Bacteroidetes and Proteobacteria. At genus level, 109 genera were found. Shannon metric for alpha diversity showed that factors like type of chickens (Commercial or experimental) and 16S rRNA gene subregion have negligible effect on diversity. Despite the large number of parameters that were taken into account, the identification of common bacteria showed five genera to be common for all sets in at least 50% of the samples. These genera are highly associated to cellulose degradation and short chain fatty acids synthesis. In general, it was possible to identify some commonalities in the bacterial cecal microbial community despite the extensive variability and factors differing from one study to another.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="hufsky_computational_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1093/bib/bbaa232" rel="external nofollow noopener">Computational strategies to combat COVID-19: useful tools to accelerate SARS-CoV-2 and coronavirus research</a></div> <div class="author"> Franziska Hufsky, Kevin Lamkiewicz, Alexandre Almeida, and <span class="more-authors" title="click to view 52 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '52 more authors' ? 'Abdel Aouacheria, Cecilia Arighi, Alex Bateman, Jan Baumbach, Niko Beerenwinkel, Christian Brandt, Marco Cacciabue, Sara Chuguransky, Oliver Drechsel, Robert D Finn, Adrian Fritz, Stephan Fuchs, Georges Hattab, Anne-Christin Hauschild, Dominik Heider, Marie Hoffmann, Martin Hölzer, Stefan Hoops, Lars Kaderali, Ioanna Kalvari, Max Kleist, Renó Kmiecinski, Denise Kühnert, Gorka Lasso, Pieter Libin, Markus List, Hannah F Löchel, Maria J Martin, Roman Martin, Julian Matschinske, Alice C McHardy, Pedro Mendes, Jaina Mistry, Vincent Navratil, Eric P Nawrocki, Áine Niamh O’Toole, Nancy Ontiveros-Palacios, Anton I Petrov, Guillermo Rangel-Pineros, Nicole Redaschi, Susanne Reimering, Knut Reinert, Alejandro Reyes, Lorna Richardson, David L Robertson, Sepideh Sadegh, Joshua B Singer, Kristof Theys, Chris Upton, Marius Welzel, Lowri Williams, Manja Marz' : '52 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">52 more authors</span> </div> <div class="periodical"> <em>Briefings in Bioinformatics</em>, Mar 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1093/bib/bbaa232" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is a novel virus of the family Coronaviridae. The virus causes the infectious disease COVID-19. The biology of coronaviruses has been studied for many years. However, bioinformatics tools designed explicitly for SARS-CoV-2 have only recently been developed as a rapid reaction to the need for fast detection, understanding and treatment of COVID-19. To control the ongoing COVID-19 pandemic, it is of utmost importance to get insight into the evolution and pathogenesis of the virus. In this review, we cover bioinformatics workflows and tools for the routine detection of SARS-CoV-2 infection, the reliable analysis of sequencing data, the tracking of the COVID-19 pandemic and evaluation of containment measures, the study of coronavirus evolution, the discovery of potential drug targets and development of therapeutic strategies. For each tool, we briefly describe its use case and how it advances research specifically for SARS-CoV-2. All tools are free to use and available online, either through web applications or public code repositories. Contact:evbc@unj-jena.de</p> </div> </div> </div> </li> <li> <div class="row"> <div id="gradaschi_genome_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1089/phage.2020.0029" rel="external nofollow noopener">Genome Sequence and Characterization of Lactobacillus casei Phage, vB_LcaM_Lbab1 Isolated from Raw Milk</a></div> <div class="author"> Victoria Gradaschi, Florencia Payaslian, Maria Eugenia Dieterle, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Liliana Rondón Salazar, Estefanía Urdániz, Matias Di Paola, José Peña Cárcamo, Fabio Zon, Mariana Allievi, Ezequiel Sosa, Darío Fernandez Do Porto, Matthew Dunne, Pauline Goeller, Jochen Klumpp, Raúl Ricardo Raya, Alejandro Reyes, Mariana Piuri' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>PHAGE</em>, Mar 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1089/phage.2020.0029" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Introduction: Only a few Lactobacillus casei phages have so far been characterized. As several L. casei strains are part of probiotic formulations, bacteriophage outbreaks targeting these strains can lead to critical losses within the dairy industry. Materials and Methods: A new L. casei phage was isolated from raw milk obtained from a milking yard from the province of Buenos Aires. The phage genome was sequenced, annotated, and analyzed. Morphology was determined by electron microscopy and the host range was established. Results:Lactobacillus phage vB_LcaM_Lbab1 is a member of the Herelleviridae family and features a host range including L. casei/Lactobacillus paracasei and Lactobacillus kefiri strains. We further analyzed the baseplate proteins in silico and found putative carbohydrate binding modules that are responsible for host recognition in other Lactobacillus phages. Conclusions: A new Lactobacillus phage was isolated and characterized. The focus was made on its host recognition mechanism, pointing toward the development of future strategies to avoid deleterious infections in the dairy industry.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="payaslian_tightening_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1089/phage.2020.0028" rel="external nofollow noopener">Tightening Bonds in Latin America Through Phage Discovery</a></div> <div class="author"> Florencia Payaslian, Victoria Gradaschi, Liliana Rondón Salazar, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Maria Eugenia Dieterle, Estefanía Urdániz, Matias Di Paola, Fabio Zon, Mariana Allievi, Carmen Sanchez Rivas, Raúl Ricardo Raya, Alejandro Reyes, Mariana Piuri' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>PHAGE</em>, Mar 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1089/phage.2020.0028" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Between 2015 and 2019, we hosted an International Phage Course at Facultad de Ciencias Exactas y Naturales, Universidad de Buenos Aires, Argentina. The 2-week full-time course was hands-on and included lectures from renowned phage biologists. Participating students were able to meet and discuss with recognized experts from around the world in a familiar setting, facilitating the establishment of scientific collaborations and the expansion of their networks. Eighty-four students from 14 Latin American countries have participated in the course, which included isolation, characterization, genome sequencing, and annotation of novel phages. We have successfully created a coursework that enabled the acquisition of new knowledge and expertise in bacteriophage biology and strengthened ties among Latin American colleagues.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="payaslian_isolation_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1089/phage.2020.0030" rel="external nofollow noopener">Isolation and Characterization of vB_MsmS_Celfi: A New Mycobacterium tuberculosis Bacteriophage</a></div> <div class="author"> Florencia Payaslian, Victoria Gradaschi, Liliana Rondón Salazar, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Maria Eugenia Dieterle, Estefanía Urdániz, Matias Di Paola, José Peña Cárcamo, Fabio Zon, Ezequiel Sosa, Darío Fernandez Do Porto, Matthew Dunne, Pauline Goeller, Jochen Klumpp, Raúl Ricardo Raya, Alejandro Reyes, Mariana Piuri' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>PHAGE</em>, Mar 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1089/phage.2020.0030" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Introduction: Because of the clinical relevance of Mycobacteria, and from a therapeutic perspective, there is an increasing interest to study phages that infect bacteria belonging to this genus. Materials and Methods: A phage was isolated from a soil sample, using Mycobacterium smegmatis as host. Its characterization included sequencing, annotation, and analysis of the genome, host range determination, and electron microscopy imaging. Results:Mycobacterium phage vB_MsmS_Celfi is a temperate phage able to infect Mycobacterium tuberculosis with high efficiency. From electron microscopy images, Celfi belongs to the Siphoviridae family. Genome analysis classified phage Celfi into cluster L, subcluster L2 of Actinobacteriophage clusters. Mycobacterium phage Celfi exhibits a Lysin B distant to those present in other members of the subcluster and other mycobacteriophages. Conclusions: The discovery of new phages that infect M. tuberculosis could contribute to the development of novel tools for detection systems and future treatment of the disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="quiroga-gonzalez_monitoring_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/s41598-021-85990-0" rel="external nofollow noopener">Monitoring the variation in the gut microbiota of captive woolly monkeys related to changes in diet during a reintroduction process</a></div> <div class="author"> Camilo Quiroga-González, Luis Alberto Chica Cardenas, Mónica Ramírez, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Alejandro Reyes, Camila González, Pablo R. Stevenson' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Scientific Reports</em>, Mar 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/s41598-021-85990-0" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Microbiome is known to play an important role in the health of organisms and different factors such as diet have been associated with modifications in microbial communities. Differences in the microbiota composition of wild and captive animals has been evaluated; however, variation during a reintroduction process in primates has never been reported. Our aim was to identify changes in the bacterial composition of three individuals of reintroduced woolly monkeys (Lagothrix lagothricha) and the variables associated with such changes. Fecal samples were collected and the V4 region of the 16S rRNA gene was sequenced to determine gut microbial composition and functionality. Individual samples from released individuals showed a higher microbial diversity after being released compared to before liberation, associated with changes in their diet. Beta diversity and functionality analysis showed separation of samples from released and captive conditions and the major factor of variation was the moment of liberation. This study shows that intestinal microbiota varies depending on site conditions and is mainly associated with diet diversity. The intake of food from wild origin by released primates may promote a positive effect on gut microbiota, improving health, and potentially increasing success in reintroduction processes.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="lopez-leal_novel_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1007/s00705-021-05010-4" rel="external nofollow noopener">A novel vieuvirus from multidrug-resistant Acinetobacter baumannii</a></div> <div class="author"> Gamaliel López-Leal, Alejandro Reyes-Muñoz, Rosa Isela Santamaria, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Miguel A. Cevallos, Carlos Pérez-Monter, Santiago Castillo-Ramírez' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Archives of Virology</em>, May 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1007/s00705-021-05010-4" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Bacteriophages are considered the most abundant biological entities on earth, and they are able to modulate the populations of their bacterial hosts. Although the potential of bacteriophages has been accepted as an alternative strategy to combat multidrug-resistant pathogenic bacteria, there still exists a considerable knowledge gap regarding their genetic diversity, which hinders their use as antimicrobial agents. In this study, we undertook a genomic and phylogenetic characterization of the phage Ab11510-phi, which was isolated from a multidrug-resistant Acinetobacter baumannii strain (Ab11510). We found that Ab11510-phi has a narrow host range and belongs to a small group of transposable phages of the genus Vieuvirus that have only been reported to infect Acinetobacter bacteria. Finally, we showed that Ab11510-phi (as well as other vieuvirus phages) has a high level of mosaicism. On a broader level, we demonstrate that comparative genomics and phylogenetic analysis are necessary tools for the proper characterization of phage diversity.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="vasquez_discovery_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1080/07391102.2020.1763839" rel="external nofollow noopener">Discovery of new potential CDK2/VEGFR2 type II inhibitors by fragmentation and virtual screening of natural products</a></div> <div class="author"> Andrés Felipe Vásquez, Alejandro Reyes Muñoz, Jorge Duitama, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Andrés González Barrios' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Journal of Biomolecular Structure and Dynamics</em>, Jun 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1080/07391102.2020.1763839" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Cyclin-Dependent Kinase 2 (CDK2) and Vascular Endothelial Growth Factor Receptor (VEGFR2) have largely been considered as attractive targets for developing anticancer agents. However, there is no dual inhibitor commercially available in the market that interacts simultaneously with the allosteric back pocket of these enzymes. We applied a combined computational strategy that started with the generation of two overlapping pharmacophore models of both kinases at ‘inactive’ conformation. Next, several virtual libraries of natural products, including the databases TCM (Traditional Chinese Medicine), UEFS (Universidade Estadual de Feira de Santana), NuBBE (Nuclei of Bioassays, Biosynthesis, and Ecophysiology of Natural Products) and AfroDb (African Medicinal Plants Database) were deconstructed using a non-extensive version of the approach RECAP (retrosynthetic combinatorial analysis procedure). These natural-product-derived fragments (NPDFs) were screened and merged into drug-sized compounds, which were filtered by Lipinski’s Rule-of-five (Ro5) and docking. As a result, two pharmacophore models, namely Hypo1 and Hypo2, were developed with an accuracy of 0.94 and 0.84, respectively. Deconstruction of natural products produced a set of 16655 unique non-extensive NPDFs that were screened against both pharmacophore models. Finally, after merging, Ro5-filtering and docking, we obtained a set of 20 hit compounds predicted to be diverse, developable, synthesizable and potent. The computational strategy proved successful to find virtual candidates of kinase inhibitors and therefore contributes to the identification of innovative multi-target compounds with potential anticancer activity. Communicated by Ramaswamy H. Sarma</p> </div> </div> </div> </li> <li> <div class="row"> <div id="moreno-gallego_informative_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3390/v13061164" rel="external nofollow noopener">Informative Regions In Viral Genomes</a></div> <div class="author"> Jaime Leonardo Moreno-Gallego, and Alejandro Reyes </div> <div class="periodical"> <em>Viruses</em>, Jun 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3390/v13061164" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Viruses, far from being just parasites affecting hosts’ fitness, are major players in any microbial ecosystem. In spite of their broad abundance, viruses, in particular bacteriophages, remain largely unknown since only about 20% of sequences obtained from viral community DNA surveys could be annotated by comparison with public databases. In order to shed some light into this genetic dark matter we expanded the search of orthologous groups as potential markers to viral taxonomy from bacteriophages and included eukaryotic viruses, establishing a set of 31,150 ViPhOGs (Eukaryotic Viruses and Phages Orthologous Groups). To do this, we examine the non-redundant viral diversity stored in public databases, predict proteins in genomes lacking such information, and used all annotated and predicted proteins to identify potential protein domains. The clustering of domains and unannotated regions into orthologous groups was done using cogSoft. Finally, we employed a random forest implementation to classify genomes into their taxonomy and found that the presence or absence of ViPhOGs is significantly associated with their taxonomy. Furthermore, we established a set of 1457 ViPhOGs that given their importance for the classification could be considered as markers or signatures for the different taxonomic groups defined by the ICTV at the order, family, and genus levels.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="krupovic_bacterial_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1007/s00705-021-05205-9" rel="external nofollow noopener">Bacterial Viruses Subcommittee and Archaeal Viruses Subcommittee of the ICTV: update of taxonomy changes in 2021</a></div> <div class="author"> Mart Krupovic, Dann Turner, Vera Morozova, and <span class="more-authors" title="click to view 46 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '46 more authors' ? 'Mike Dyall-Smith, Hanna M. Oksanen, Rob Edwards, Bas E. Dutilh, Susan M. Lehman, Alejandro Reyes, Diana P. Baquero, Matthew B. Sullivan, Jumpei Uchiyama, Jesca Nakavuma, Jakub Barylski, Mark J. Young, Shishen Du, Poliane Alfenas-Zerbini, Alla Kushkina, Andrew M. Kropinski, Ipek Kurtböke, J. Rodney Brister, Cédric Lood, B. L. Sarkar, Tong Yigang, Ying Liu, Li Huang, Johannes Wittmann, Nina Chanishvili, Leonardo J. Zyl, Janis Rumnieks, Tomohiro Mochizuki, Matti Jalasvuori, Ramy K. Aziz, Małgorzata Łobocka, Kenneth M. Stedman, Andrey N. Shkoporov, Annika Gillis, Xu Peng, François Enault, Petar Knezevic, Rob Lavigne, Sung-Keun Rhee, Virginija Cvirkaite-Krupovic, Cristina Moraru, Andrea I. Moreno Switt, Minna M. Poranen, Andrew Millard, David Prangishvili, Evelien M. Adriaenssens' : '46 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">46 more authors</span> </div> <div class="periodical"> <em>Archives of Virology</em>, Nov 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1007/s00705-021-05205-9" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>In this article, we – the Bacterial Viruses Subcommittee and the Archaeal Viruses Subcommittee of the International Committee on Taxonomy of Viruses (ICTV) – summarise the results of our activities for the period March 2020 – March 2021. We report the division of the former Bacterial and Archaeal Viruses Subcommittee in two separate Subcommittees, welcome new members, a new Subcommittee Chair and Vice Chair, and give an overview of the new taxa that were proposed in 2020, approved by the Executive Committee and ratified by vote in 2021. In particular, a new realm, three orders, 15 families, 31 subfamilies, 734 genera and 1845 species were newly created or redefined (moved/promoted).</p> </div> </div> </div> </li> <li> <div class="row"> <div id="ayala-usma_whole_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1186/s12864-021-08079-y" rel="external nofollow noopener">A whole genome duplication drives the genome evolution of Phytophthora betacei, a closely related species to Phytophthora infestans</a></div> <div class="author"> David A. Ayala-Usma, Martha Cárdenas, Romain Guyot, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Maryam Chaib De Mares, Adriana Bernal, Alejandro Reyes Muñoz, Silvia Restrepo' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>BMC Genomics</em>, Nov 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1186/s12864-021-08079-y" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Pathogens of the genus Phytophthora are the etiological agents of many devastating diseases in several high-value crops and forestry species such as potato, tomato, cocoa, and oak, among many others. Phytophthora betacei is a recently described species that causes late blight almost exclusively in tree tomatoes, and it is closely related to Phytophthora infestans that causes the disease in potato crops and other Solanaceae. This study reports the assembly and annotation of the genomes of P. betacei P8084, the first of its species, and P. infestans RC1-10, a Colombian strain from the EC-1 lineage, using long-read SMRT sequencing technology.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="restrepo_microbial_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1186/s40168-021-01043-8" rel="external nofollow noopener">Microbial community characterization of shrimp survivors to AHPND challenge test treated with an effective shrimp probiotic (Vibrio diabolicus)</a></div> <div class="author"> Leda Restrepo, Cristóbal Domínguez-Borbor, Leandro Bajaña, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Irma Betancourt, Jenny Rodríguez, Bonny Bayot, Alejandro Reyes' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Microbiome</em>, Apr 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1186/s40168-021-01043-8" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Acute hepatopancreatic necrosis disease (AHPND) is an important shrimp bacterial disease caused by some Vibrio species. The severity of the impact of this disease on aquaculture worldwide has made it necessary to develop alternatives to prophylactic antibiotics use, such as the application of probiotics. To assess the potential to use probiotics in order to limit the detrimental effects of AHNPD, we evaluated the effect of the ILI strain, a Vibrio sp. bacterium and efficient shrimp probiotic, using metabarcoding (16S rRNA gene) on the gastrointestinal microbiota of shrimp after being challenged with AHPND-causing V. parahaemolyticus.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="dutilh_perspective_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1016/j.coviro.2021.10.011" rel="external nofollow noopener">Perspective on taxonomic classification of uncultivated viruses</a></div> <div class="author"> Bas E Dutilh, Arvind Varsani, Yigang Tong, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Peter Simmonds, Sead Sabanadzovic, Luisa Rubino, Simon Roux, Alejandro Reyes Muñoz, Cédric Lood, Elliot J Lefkowitz, Jens H Kuhn, Mart Krupovic, Robert A Edwards, J Rodney Brister, Evelien M Adriaenssens, Matthew B Sullivan' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>Current Opinion in Virology</em>, Dec 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1016/j.coviro.2021.10.011" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Historically, virus taxonomy has been limited to describing viruses that were readily cultivated in the laboratory or emerging in natural biomes. Metagenomic analyses, single-particle sequencing, and database mining efforts have yielded new sequence data on an astounding number of previously unknown viruses. As metagenomes are relatively free of biases, these data provide an unprecedented insight into the vastness of the virosphere, but to properly value the extent of this diversity it is critical that the viruses are taxonomically classified. Inclusion of uncultivated viruses has already improved the process as well as the understanding of the taxa, viruses, and their evolutionary relationships. The continuous development and testing of computational tools will be required to maintain a dynamic virus taxonomy that can accommodate the new discoveries.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="rangel-pineros_trees_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1089/phage.2021.0008" rel="external nofollow noopener">From Trees to Clouds: PhageClouds for Fast Comparison of ∼640,000 Phage Genomic Sequences and Host-Centric Visualization Using Genomic Network Graphs</a></div> <div class="author"> Guillermo Rangel-Pineros, Andrew Millard, Slawomir Michniewski, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'David Scanlan, Kimmo Sirén, Alejandro Reyes, Bent Petersen, Martha R.J. Clokie, Thomas Sicheritz-Pontén' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>PHAGE</em>, Dec 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1089/phage.2021.0008" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Background: Fast and computationally efficient strategies are required to explore genomic relationships within an increasingly large and diverse phage sequence space. Here, we present PhageClouds, a novel approach using a graph database of phage genomic sequences and their intergenomic distances to explore the phage genomic sequence space. Methods: A total of 640,000 phage genomic sequences were retrieved from a variety of databases and public virome assemblies. Intergenomic distances were calculated with dashing, an alignment-free method suitable for handling massive data sets. These data were used to build a Neo4j® graph database. Results: PhageClouds supported the search of related phages among all complete phage genomes from GenBank for a single query phage in just 10 s. Moreover, PhageClouds expanded the number of closely related phage sequences detected for both finished and draft phage genomes, in comparison with searches exclusively targeting phage entries from GenBank. Conclusions: PhageClouds is a novel resource that will facilitate the analysis of phage genomic sequences and the characterization of assembled phage genomes.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="moore_ten_2021" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1371/journal.pcbi.1009218" rel="external nofollow noopener">Ten simple rules for organizing a bioinformatics training course in low- and middle-income countries</a></div> <div class="author"> Benjamin Moore, Patricia Carvajal-López, Paballo Abel Chauke, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Marco Cristancho, Victoria Dominguez Del Angel, Selene L. Fernandez-Valverde, Amel Ghouila, Piraveen Gopalasingam, Fatma Zahra Guerfali, Alice Matimba, Sarah L. Morgan, Guilherme Oliveira, Verena Ras, Alejandro Reyes, Javier De Las Rivas, Nicola Mulder' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>PLOS Computational Biology</em>, Aug 2021 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1371/journal.pcbi.1009218" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div id="pena_gonzalez_essentials_2020" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.7490/f1000research.1118334.1" rel="external nofollow noopener">Essentials in Metagenomics (Part II)</a></div> <div class="author"> Angela Peña González, and Alejandro Reyes Muñoz </div> <div class="periodical"> <em>F1000Research</em>, Oct 2020 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.7490/f1000research.1118334.1" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>With the development of high-throughput next generation sequencing technologies and the technical advances to generate high-quality sequencing data, the bottleneck in implementing metagenomics for economic and social purposes in Latin America has shifted from obtaining DNA sequences to post-sequencing data analysis. Latin American countries still face challenges to develop and implement strategies for local data processing and data mining due to the limited bioinformatics capacity. Although many large computing grids already exist, outsourcing analyses and computing is not always the best solution since data transfer requires significant bandwidth due to the file sizes, and more importantly, it limits the training of local researchers to perform cutting-edge bioinformatics. Therefore, developing learning material in data processing and analysis in metagenomics is critical to the formation of new highly-skilled bioinformaticians. The present document is a tutorial in essential aspects of metagenomics data processing (post- sequencing) which was structured based on an extensive literature review, with the goal of presenting the current bioinformatics tools and workflows for extracting relevant, biological information out of a large sequencing data set. This is the second of a two-part series of documents designed to provide the essential background (experimental and computational) for users to approach the field of metagenomics. Users will find an overview of the different bioinformatic analyses that are commonly performed in metagenomic studies, including quality control, decontamination, coverage estimation, assembly, recovery of MAGs (metagenome- assembled genomes), taxonomic and functional classification of reads and contigs, and strain- level comparative analysis, among many other relevant topics. Users will also find rich discussions on the advantages and limitations that the different tools and methods offer and will be provided with external links and websites where additional information can be found.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="pena_gonzalez_essentials_2020-1" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.7490/f1000research.1118333.1" rel="external nofollow noopener">Essentials in metagenomics (Part I)</a></div> <div class="author"> Angela Peña González, and Alejandro Reyes Muñoz </div> <div class="periodical"> <em>F1000Research</em>, Oct 2020 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.7490/f1000research.1118333.1" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Microbial communities play central roles in many aspects of life on earth including global geochemical cycles, natural and engineered biosystems, and health and disease of humans, animals, and plants. Metagenomics has revolutionized the study of microbes by paving the way for cultivation-independent assessments of microbial communities present in these complex ecosystems. In this sense, metagenomics can have (and is in fact having) a profound impact in several economic and social aspects of developing and developed countries, including the global surveillance of infectious diseases, disease diagnosis and treatments, food production and the protection of biodiversity. Therefore, the availability of learning material in metagenomics is critical to facilitate public health and economic development in the Americas. The present document is a guide to essential aspects of metagenomics, designed based on an extensive literature review, with the goal of presenting current principles and concepts in experimental design, sequencing technologies, data processing and statistical analysis applied to metagenomic studies in different areas. Equally important, we provide practical guidance for users with basic or no experience in the field. Users will find a concise but comprehensive introduction to the field, with important discussions and considerations to keep in mind when designing an experiment. The discussions include aspects related to sample processing, DNA extraction methods, and the technical challenges and sources of variation that must be controlled within the experimental design. Users will also find valuable information in the technical aspects behind second (Illumina/Solexa) and third generation (PacBio and Oxford Nanopore) sequencing technologies and library preparation as well as discussions on how these technologies are changing the current practices in the field of metagenomics.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="pacheco-montealegre_fine_2020" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fmicb.2020.00650" rel="external nofollow noopener">Fine Resolution Analysis of Microbial Communities Provides Insights Into the Variability of Cocoa Bean Fermentation</a></div> <div class="author"> Mauricio Edilberto Pacheco-Montealegre, Lizeth Lorena Dávila-Mora, Lina Marcela Botero-Rute, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Alejandro Reyes, Alejandro Caro-Quintero' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Oct 2020 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2020.00650" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Cocoa bean fermentation is an important microbial process, where most metabolites that affect chocolate quality and aroma are generated. Production of reproducible high-quality beans is a major challenge because most fermentations occur in open containers with a lack of variable control. Here we present a study that aims to identify the effect of farm protocols, climate, and bean mass exposure, in the dynamics and composition of microbial communities. Using high-throughput sequencing of molecular markers for bacteria and yeasts, complemented with culture-based methods, we evaluated the microbial diversity and dynamics associated to spontaneous cocoa fermentation in two distinct agro-ecological zones in Colombia. The bacterial communities were classified at two levels of evolutionary relationship, at a coarse resolution (OTU-level) and at a finer resolution (oligotype-level). A total of six bacterial OTUs were present in both farms, following a microbial succession that starts with the Enterobacteraceae family (one OTU), transitioning to the Lactobacillaceae family (three OTUs), and finishing with Acetobacteraceae family (two OTUs). When undesirable practices were done, OTUs were observed at unexpected moments during the fermentation. At a finer taxonomic resolution, 48 oligotypes were identified, with 46 present in both farms. These oligotypes have different patterns of prevalence. In the case of Lactobacillaceae a high evenness was observed among oligotypes. In contrast, for Enterobacteraceae and Acetobacteraceae a high dominance of one or two oligotypes was observed, these oligotypes were the same for both farms, despite geographic location and season of sampling. When the overall fermentations were compared using correlations matrices of oligotypes abundance, they show a clear clustering by farm, suggesting that farm protocols generate a unique fingerprint in the dynamics and interactions of the microbial communities. The comparison between the upper and middle layers of the bean mass showed that environmental exposure affects the paces at which ecological successions occur, and therefore, is an important source of cocoa quality heterogeneity. In conclusion, the results presented here showed that the dynamics of microbial fermentation can be used to identify the sources of variability and evidence the need for better fermentation technologies that favor the production of reproducible high-quality cocoa beans.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="botero_genome-scale_2020" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fgene.2020.00837" rel="external nofollow noopener">Genome-Scale Metabolic Model of Xanthomonas phaseoli pv. manihotis: An Approach to Elucidate Pathogenicity at the Metabolic Level</a></div> <div class="author"> David Botero, Jonathan Monk, María Juliana Rodríguez Cubillos, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Andrés Rodríguez Cubillos, Mariana Restrepo, Vivian Bernal-Galeano, Alejandro Reyes, Andrés González Barrios, Bernhard Ø. Palsson, Silvia Restrepo, Adriana Bernal' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Frontiers in Genetics</em>, Oct 2020 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fgene.2020.00837" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Xanthomonas phaseoli pv. manihotis (Xpm) is the causal agent of cassava bacterial blight, the most important bacterial disease in this crop. There is a paucity of knowledge about the metabolism of Xanthomonas and its relevance in the pathogenic process, with the exception of the elucidation of the xanthan biosynthesis route. Here we report the reconstruction of the genome-scale model of Xpm metabolism and the insights it provides into plant–pathogen interactions. The model, iXpm1556, displayed 1,556 reactions, 1,527 compounds, and 890 genes. Metabolic maps of central amino acid and carbohydrate metabolism, as well as xanthan biosynthesis of Xpm, were reconstructed using Escher (https://escher.github.io/) to guide the curation process and for further analyses. The model was constrained using the RNA-seq data of a mutant of Xpm for quorum sensing (QS), and these data were used to construct context-specific models (CSMs) of the metabolism of the two strains (wild type and QS mutant). The CSMs and flux balance analysis were used to get insights into pathogenicity, xanthan biosynthesis, and QS mechanisms. Between the CSMs, 653 reactions were shared; unique reactions belong to purine, pyrimidine, and amino acid metabolism. Alternative objective functions were used to demonstrate a trade-off between xanthan biosynthesis and growth and the re-allocation of resources in the process of biosynthesis. Important features altered by QS included carbohydrate metabolism, NAD(P)+ balance, and fatty acid elongation. In this work, we modeled the xanthan biosynthesis and the QS process and their impact on the metabolism of the bacterium. This model will be useful for researchers studying host–pathogen interactions and will provide insights into the mechanisms of infection used by this and other Xanthomonas species.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="keane_international_2020" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1111/codi.14957" rel="external nofollow noopener">International consensus definition of low anterior resection syndrome</a></div> <div class="author"> C. Keane, N. S. Fearnhead, L. Bordeianou, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'P. Christensen, E. Espin Basany, S. Laurberg, A. Mellgren, C. Messick, G. R. Orangio, A. Verjee, K. Wing, I. Bissett, the LARS International Collaborative Group' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>Colorectal Disease</em>, Oct 2020 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1111/codi.14957" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Aim Low anterior resection syndrome (LARS) is pragmatically defined as disordered bowel function after rectal resection leading to a detriment in quality of life. This broad characterization does not allow for precise estimates of prevalence. The LARS score was designed as a simple tool for clinical evaluation of LARS. Although the LARS score has good clinical utility, it may not capture all important aspects that patients may experience. The aim of this collaboration was to develop an international consensus definition of LARS that encompasses all aspects of the condition and is informed by all stakeholders. Method This international patient–provider initiative used an online Delphi survey, regional patient consultation meetings, and an international consensus meeting. Three expert groups participated: patients, surgeons and other health professionals from five regions (Australasia, Denmark, Spain, Great Britain and Ireland, and North America) and in three languages (English, Spanish, and Danish). The primary outcome measured was the priorities for the definition of LARS. Results Three hundred twenty-five participants (156 patients) registered. The response rates for successive rounds of the Delphi survey were 86%, 96% and 99%. Eighteen priorities emerged from the Delphi survey. Patient consultation and consensus meetings refined these priorities to eight symptoms and eight consequences that capture essential aspects of the syndrome. Sampling bias may have been present, in particular, in the patient panel because social media was used extensively in recruitment. There was also dominance of the surgical panel at the final consensus meeting despite attempts to mitigate this. Conclusion This is the first definition of LARS developed with direct input from a large international patient panel. The involvement of patients in all phases has ensured that the definition presented encompasses the vital aspects of the patient experience of LARS. The novel separation of symptoms and consequences may enable greater sensitivity to detect changes in LARS over time and with intervention.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="reyes_paving_2020" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3390/v12090905" rel="external nofollow noopener">Paving the Way to Unveil the Diversity and Evolution of Phage Genomes</a></div> <div class="author"> Alejandro Reyes, and Martha J. Vives </div> <div class="periodical"> <em>Viruses</em>, Sep 2020 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3390/v12090905" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Phage biology has been developing for the last hundred years, and the potential of phages as tools and treatments has been known since their early discovery. However, the lack of knowledge of the molecular mechanisms coded in phage genomes hindered the development of the field. With current molecular methods, the last decade has been a resurgence of the field. The Special Issue on “Diversity and Evolution of Phage Genomes” is a great example with its 17 manuscripts published. It covers some of the latest methods to sample and characterize environmental and host associated viromes, considering experimental biases and computational developments. Furthermore, the use of molecular tools coupled with traditional methods has allowed to isolate and characterize viruses from different hosts and environments with such diversity that even a new viral class is being proposed. The viruses described cover all different phage families and lifestyles. However, is not only about diversity; the molecular evolution is studied in a set of manuscripts looking at phage-host interactions and their capacity to uncover the frequency and type of mutations behind the bacterial resistance mechanisms and viral pathogenesis, and such methods are opening new ways into identifying potential receptors and characterizing the bacterial host range.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="garcia-vega_diet_2020" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3390/nu12102938" rel="external nofollow noopener">Diet Quality, Food Groups and Nutrients Associated with the Gut Microbiota in a Nonwestern Population</a></div> <div class="author"> Ángela S. García-Vega, Vanessa Corrales-Agudelo, Alejandro Reyes, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Juan S. Escobar' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Nutrients</em>, Oct 2020 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3390/nu12102938" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Diet plays an important role in shaping gut microbiota. However, much remains to be learned regarding this association. We analyzed dietary intake and gut microbiota in a community-dwelling cohort of 441 Colombians. Diet quality, intake of food groups and nutrient consumption were paired with microbial diversity and composition using linear regressions, Procrustes analyses and a random-forest machine-learning algorithm. Analyses were adjusted for potential confounders, including the five cities from where the participants originated, sex (male, female), age group (18–40 and 41–62 years), BMI (lean, overweight, obese) and socioeconomic status. Microbial diversity was higher in individuals with increased intake of nutrients obtained from plant-food sources, whereas the intake of food groups and nutrients correlated with microbiota structure. Random-forest regressions identified microbial communities associated with different diet components. Two remarkable results confirmed previous expectations regarding the link between diet and microbiota: communities composed of short-chain fatty acid (SCFA) producers were more prevalent in the microbiota of individuals consuming diets rich in fiber and plant-food sources, such as fruits, vegetables and beans. In contrast, an inflammatory microbiota composed of bile-tolerant and putrefactive microorganisms along with opportunistic pathogens thrived in individuals consuming diets enriched in animal-food sources and of low quality, i.e., enriched in ultraprocessed foods and depleted in dietary fiber. This study expands our understanding of the relationship between dietary intake and gut microbiota. We provide evidence that diet is strongly associated with the gut microbial community and highlight generalizable connections between them.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div id="roux_minimum_2019" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/nbt.4306" rel="external nofollow noopener">Minimum Information about an Uncultivated Virus Genome (MIUViG)</a></div> <div class="author"> Simon Roux, Evelien M. Adriaenssens, Bas E. Dutilh, and <span class="more-authors" title="click to view 58 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '58 more authors' ? 'Eugene V. Koonin, Andrew M. Kropinski, Mart Krupovic, Jens H. Kuhn, Rob Lavigne, J. Rodney Brister, Arvind Varsani, Clara Amid, Ramy K. Aziz, Seth R. Bordenstein, Peer Bork, Mya Breitbart, Guy R. Cochrane, Rebecca A. Daly, Christelle Desnues, Melissa B. Duhaime, Joanne B. Emerson, François Enault, Jed A. Fuhrman, Pascal Hingamp, Philip Hugenholtz, Bonnie L. Hurwitz, Natalia N. Ivanova, Jessica M. Labonté, Kyung-Bum Lee, Rex R. Malmstrom, Manuel Martinez-Garcia, Ilene Karsch Mizrachi, Hiroyuki Ogata, David Páez-Espino, Marie-Agnès Petit, Catherine Putonti, Thomas Rattei, Alejandro Reyes, Francisco Rodriguez-Valera, Karyna Rosario, Lynn Schriml, Frederik Schulz, Grieg F. Steward, Matthew B. Sullivan, Shinichi Sunagawa, Curtis A. Suttle, Ben Temperton, Susannah G. Tringe, Rebecca Vega Thurber, Nicole S. Webster, Katrine L. Whiteson, Steven W. Wilhelm, K. Eric Wommack, Tanja Woyke, Kelly C. Wrighton, Pelin Yilmaz, Takashi Yoshida, Mark J. Young, Natalya Yutin, Lisa Zeigler Allen, Nikos C. Kyrpides, Emiley A. Eloe-Fadrosh' : '58 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">58 more authors</span> </div> <div class="periodical"> <em>Nature Biotechnology</em>, Jan 2019 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/nbt.4306" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We present an extension of the Minimum Information about any (x) Sequence (MIxS) standard for reporting sequences of uncultivated virus genomes. Minimum Information about an Uncultivated Virus Genome (MIUViG) standards were developed within the Genomic Standards Consortium framework and include virus origin, genome quality, genome annotation, taxonomic classification, biogeographic distribution and in silico host prediction. Community-wide adoption of MIUViG standards, which complement the Minimum Information about a Single Amplified Genome (MISAG) and Metagenome-Assembled Genome (MIMAG) standards for uncultivated bacteria and archaea, will improve the reporting of uncultivated virus genomes in public databases. In turn, this should enable more robust comparative studies and a systematic exploration of the global virosphere.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="amann_toward_2019" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1126/science.aaw1280" rel="external nofollow noopener">Toward unrestricted use of public genomic data</a></div> <div class="author"> Rudolf I. Amann, Shakuntala Baichoo, Benjamin J. Blencowe, and <span class="more-authors" title="click to view 47 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '47 more authors' ? 'Peer Bork, Mark Borodovsky, Cath Brooksbank, Patrick S. G. Chain, Rita R. Colwell, Daniele G. Daffonchio, Antoine Danchin, Victor Lorenzo, Pieter C. Dorrestein, Robert D. Finn, Claire M. Fraser, Jack A. Gilbert, Steven J. Hallam, Philip Hugenholtz, John P. A. Ioannidis, Janet K. Jansson, Jihyun F. Kim, Hans-Peter Klenk, Martin G. Klotz, Rob Knight, Konstantinos T. Konstantinidis, Nikos C. Kyrpides, Christopher E. Mason, Alice C. McHardy, Folker Meyer, Christos A. Ouzounis, Aristides A. N. Patrinos, Mircea Podar, Katherine S. Pollard, Jacques Ravel, Alejandro Reyes Muñoz, Richard J. Roberts, Ramon Rosselló-Móra, Susanna-Assunta Sansone, Patrick D. Schloss, Lynn M. Schriml, João C. Setubal, Rotem Sorek, Rick L. Stevens, James M. Tiedje, Adrian Turjanski, Gene W. Tyson, David W. Ussery, George M. Weinstock, Owen White, William B. Whitman, Ioannis Xenarios' : '47 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">47 more authors</span> </div> <div class="periodical"> <em>Science</em>, Jan 2019 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1126/science.aaw1280" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Despite some notable progress in data sharing policies and practices, restrictions are still often placed on the open and unconditional use of various genomic data after they have received official approval for release to the public domain or to public databases. These restrictions, which often conflict with the terms and conditions of the funding bodies who supported the release of those data for the benefit of the scientific community and society, are perpetuated by the lack of clear guiding rules for data usage. Existing guidelines for data released to the public domain recognize but fail to resolve tensions between the importance of free and unconditional use of these data and the “right” of the data producers to the first publication. This self-contradiction has resulted in a loophole that allows different interpretations and a continuous debate between data producers and data users on the use of public data. We argue that the publicly available data should be treated as open data, a shared resource with unrestricted use for analysis, interpretation, and publication.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="moreno-gallego_virome_2019" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1016/j.chom.2019.01.019" rel="external nofollow noopener">Virome Diversity Correlates with Intestinal Microbiome Diversity in Adult Monozygotic Twins</a></div> <div class="author"> J. Leonardo Moreno-Gallego, Shao-Pei Chou, Sara C. Di Rienzi, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Julia K. Goodrich, Timothy D. Spector, Jordana T. Bell, Nicholas D. Youngblut, Ian Hewson, Alejandro Reyes, Ruth E. Ley' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Cell Host &amp; Microbe</em>, Feb 2019 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1016/j.chom.2019.01.019" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The virome is one of the most variable components of the human gut microbiome. Within twin pairs, viromes have been shown to be similar for infants, but not for adults, indicating that as twins age and their environments and microbiomes diverge, so do their viromes. The degree to which the microbiome drives the vast virome diversity is unclear. Here, we examine the relationship between microbiome and virome diversity in 21 adult monozygotic twin pairs selected for high or low microbiome concordance. Viromes derived from virus-like particles are unique to each individual, are dominated by Caudovirales and Microviridae, and exhibit a small core that includes crAssphage. Microbiome-discordant twins display more dissimilar viromes compared to microbiome-concordant twins, and the richer the microbiomes, the richer the viromes. These patterns are driven by bacteriophages, not eukaryotic viruses. Collectively, these observations support a strong role of the microbiome in patterning for the virome.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="andrade-martinez_defining_2019" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/s41598-019-47742-z" rel="external nofollow noopener">Defining a Core Genome for the Herpesvirales and Exploring their Evolutionary Relationship with the Caudovirales</a></div> <div class="author"> Juan S. Andrade-Martínez, J. Leonardo Moreno-Gallego, and Alejandro Reyes </div> <div class="periodical"> <em>Scientific Reports</em>, Aug 2019 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/s41598-019-47742-z" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The order Herpesvirales encompasses a wide variety of important and broadly distributed human pathogens. During the last decades, similarities in the viral cycle and the structure of some of their proteins with those of the order Caudovirales, the tailed bacterial viruses, have brought speculation regarding the existence of an evolutionary relationship between these clades. To evaluate such hypothesis, we used over 600 Herpesvirales and 2000 Caudovirales complete genomes to search for the presence or absence of clusters of orthologous protein domains and constructed a dendrogram based on their compositional similarities. The results obtained strongly suggest an evolutionary relationship between the two orders. Furthermore, they allowed to propose a core genome for the Herpesvirales, composed of 4 proteins, including the ATPase subunit of the DNA-packaging terminase, the only protein with previously verified conservation. Accordingly, a phylogenetic tree constructed with sequences derived from the clusters associated to these proteins grouped the Herpesvirales strains accordingly to the established families and subfamilies. Overall, this work provides results supporting the hypothesis that the two orders are evolutionarily related and contributes to the understanding of the history of the Herpesvirales.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="posada-perlaza_bogota_2019" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/s41598-019-48200-6" rel="external nofollow noopener">Bogotá River anthropogenic contamination alters microbial communities and promotes spread of antibiotic resistance genes</a></div> <div class="author"> Carlos Eduardo Posada-Perlaza, Adán Ramírez-Rojas, Paola Porras, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Boahemaa Adu-Oppong, Ana-María Botero-Coy, Félix Hernández, Juan M. Anzola, Lorena Díaz, Gautam Dantas, Alejandro Reyes, María Mercedes Zambrano' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Scientific Reports</em>, Aug 2019 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/s41598-019-48200-6" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The increase in antibiotic resistant bacteria has raised global concern regarding the future effectiveness of antibiotics. Human activities that influence microbial communities and environmental resistomes can generate additional risks to human health. In this work, we characterized aquatic microbial communities and their resistomes in samples collected at three sites along the Bogotá River and from wastewaters at three city hospitals, and investigated community profiles and antibiotic resistance genes (ARGs) as a function of anthropogenic contamination. The presence of antibiotics and other commonly used drugs increased in locations highly impacted by human activities, while the diverse microbial communities varied among sites and sampling times, separating upstream river samples from more contaminated hospital and river samples. Clinically relevant antibiotic resistant pathogens and ARGs were more abundant in contaminated water samples. Tracking of resistant determinants to upstream river waters and city sources suggested that human activities foster the spread of ARGs, some of which were co-localized with mobile genetic elements in assembled metagenomic contigs. Human contamination of this water ecosystem changed both community structure and environmental resistomes that can pose a risk to human health.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="edwards_global_2019" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/s41564-019-0494-6" rel="external nofollow noopener">Global phylogeography and ancient evolution of the widespread human gut virus crAssphage</a></div> <div class="author"> Robert A. Edwards, Alejandro A. Vega, Holly M. Norman, and <span class="more-authors" title="click to view 114 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '114 more authors' ? 'Maria Ohaeri, Kyle Levi, Elizabeth A. Dinsdale, Ondrej Cinek, Ramy K. Aziz, Katelyn McNair, Jeremy J. Barr, Kyle Bibby, Stan J. J. Brouns, Adrian Cazares, Patrick A. Jonge, Christelle Desnues, Samuel L. Díaz Muñoz, Peter C. Fineran, Alexander Kurilshikov, Rob Lavigne, Karla Mazankova, David T. McCarthy, Franklin L. Nobrega, Alejandro Reyes Muñoz, German Tapia, Nicole Trefault, Alexander V. Tyakht, Pablo Vinuesa, Jeroen Wagemans, Alexandra Zhernakova, Frank M. Aarestrup, Gunduz Ahmadov, Abeer Alassaf, Josefa Anton, Abigail Asangba, Emma K. Billings, Vito Adrian Cantu, Jane M. Carlton, Daniel Cazares, Gyu-Sung Cho, Tess Condeff, Pilar Cortés, Mike Cranfield, Daniel A. Cuevas, Rodrigo Iglesia, Przemyslaw Decewicz, Michael P. Doane, Nathaniel J. Dominy, Lukasz Dziewit, Bashir Mukhtar Elwasila, A. Murat Eren, Charles Franz, Jingyuan Fu, Cristina Garcia-Aljaro, Elodie Ghedin, Kristen M. Gulino, John M. Haggerty, Steven R. Head, Rene S. Hendriksen, Colin Hill, Heikki Hyöty, Elena N. Ilina, Mitchell T. Irwin, Thomas C. Jeffries, Juan Jofre, Randall E. Junge, Scott T. Kelley, Mohammadali Khan Mirzaei, Martin Kowalewski, Deepak Kumaresan, Steven R. Leigh, David Lipson, Eugenia S. Lisitsyna, Montserrat Llagostera, Julia M. Maritz, Linsey C. Marr, Angela McCann, Shahar Molshanski-Mor, Silvia Monteiro, Benjamin Moreira-Grez, Megan Morris, Lawrence Mugisha, Maite Muniesa, Horst Neve, Nam-phuong Nguyen, Olivia D. Nigro, Anders S. Nilsson, Taylor O’Connell, Rasha Odeh, Andrew Oliver, Mariana Piuri, Aaron J. Prussin II, Udi Qimron, Zhe-Xue Quan, Petra Rainetova, Adán Ramírez-Rojas, Raul Raya, Kim Reasor, Gillian A. O. Rice, Alessandro Rossi, Ricardo Santos, John Shimashita, Elyse N. Stachler, Lars C. Stene, Ronan Strain, Rebecca Stumpf, Pedro J. Torres, Alan Twaddle, MaryAnn Ugochi Ibekwe, Nicolás Villagra, Stephen Wandro, Bryan White, Andy Whiteley, Katrine L. Whiteson, Cisca Wijmenga, Maria M. Zambrano, Henrike Zschach, Bas E. Dutilh' : '114 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">114 more authors</span> </div> <div class="periodical"> <em>Nature Microbiology</em>, Oct 2019 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/s41564-019-0494-6" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Microbiomes are vast communities of microorganisms and viruses that populate all natural ecosystems. Viruses have been considered to be the most variable component of microbiomes, as supported by virome surveys and examples of high genomic mosaicism. However, recent evidence suggests that the human gut virome is remarkably stable compared with that of other environments. Here, we investigate the origin, evolution and epidemiology of crAssphage, a widespread human gut virus. Through a global collaboration, we obtained DNA sequences of crAssphage from more than one-third of the world’s countries and showed that the phylogeography of crAssphage is locally clustered within countries, cities and individuals. We also found fully colinear crAssphage-like genomes in both Old-World and New-World primates, suggesting that the association of crAssphage with primates may be millions of years old. Finally, by exploiting a large cohort of more than 1,000 individuals, we tested whether crAssphage is associated with bacterial taxonomic groups of the gut microbiome, diverse human health parameters and a wide range of dietary factors. We identified strong correlations with different clades of bacteria that are related to Bacteroidetes and weak associations with several diet categories, but no significant association with health or disease. We conclude that crAssphage is a benign cosmopolitan virus that may have coevolved with the human lineage and is an integral part of the normal human gut virome.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="diaz-riano_computational_2019" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1371/journal.pone.0221540" rel="external nofollow noopener">Computational search for UV radiation resistance strategies in Deinococcus swuensis isolated from Paramo ecosystems</a></div> <div class="author"> Jorge Díaz-Riaño, Leonardo Posada, Iván Camilo Acosta, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Carlos Ruíz-Pérez, Catalina García-Castillo, Alejandro Reyes, María Mercedes Zambrano' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>PLOS ONE</em>, Dec 2019 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1371/journal.pone.0221540" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Ultraviolet radiation (UVR) is widely known as deleterious for many organisms since it can cause damage to biomolecules either directly or indirectly via the formation of reactive oxygen species. The goal of this study was to analyze the capacity of high-mountain Espeletia hartwegiana plant phyllosphere microorganisms to survive UVR and to identify genes related to resistance strategies. A strain of Deinococcus swuensis showed a high survival rate of up to 60% after UVR treatment at 800J/m2 and was used for differential expression analysis using RNA-seq after exposing cells to 400J/m2 of UVR (with \textgreater95% survival rate). Differentially expressed genes were identified using the R-Bioconductor package NOISeq and compared with other reported resistance strategies reported for this genus. Genes identified as being overexpressed included transcriptional regulators and genes involved in protection against damage by UVR. Non-coding (nc)RNAs were also differentially expressed, some of which have not been previously implicated. This study characterized the immediate radiation response of D. swuensis and indicates the involvement of ncRNAs in the adaptation to extreme environmental conditions.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div id="gonzalez-zapata_holobiont_2018" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fmars.2018.00029" rel="external nofollow noopener">Holobiont Diversity in a Reef-Building Coral over Its Entire Depth Range in the Mesophotic Zone</a></div> <div class="author"> Fanny L. Gonzalez-Zapata, Pim Bongaerts, Catalina Ramírez-Portilla, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Boahemaa Adu-Oppong, Gretchen Walljasper, Alejandro Reyes, Juan A. Sanchez' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Frontiers in Marine Science</em>, Dec 2018 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fmars.2018.00029" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Mesophotic reef-building coral communities (~30–120 m depth) remain largely unexplored, despite representing roughly three-quarters of the overall depth range at which tropical coral reef ecosystems occur. Although many coral species are restricted to shallow depths, several species occur across large depth ranges, including lower mesophotic depths. Yet, it remains unclear how such species can persist under extreme low-light conditions and how the different symbiotic partners associated with these corals contribute to facilitate such broad depth ranges. We assessed holobiont genetic diversity of the Caribbean coral Agaricia undata over depth in three localities of Colombia: San Andres Island (between 37 and 85 m), Cartagena (between 17 and 45 m) and “Parque Nacional Natural Corales de Profundidad” (between 77 and 87 m). We used a population genomics approach (NextRAD) for the coral host, and amplicon sequencing for the associated Symbiodinium (non-coding region of the plastid psbA minicircle) and prokaryotic (V4 region of the 16S rRNA gene) symbiont community. For the coral host, genetic structuring was only observed across geographic regions, but not between depths. Bayesian clustering and discriminant analysis of principal components revealed genetic structuring between the three regions, but not between shallow (\textless30 m), upper (≥30 and ≤60 m) and lower mesophotic (\textgreater60 m) depths. This pattern was confirmed when evaluating pairwise differentiation (FST) between populations, with much higher values between regions (0.0467–0.1034) compared to between depths [within location; −0.0075–(−0.0007)]. Symbiotic partners, including seven types of zooxanthellae and 325 prokaryotic OTUs, did not exhibit partitioning across depths. All samples hosted Symbiodinium clade C3 and the type C3psbA_e was present in all depths. Alpha microbial diversity was not significantly different between zones (upper vs. lower), which community composition between coral colonies was similar in the two zones (ANOSIM, R = −0.079, P \textgreater 0.05). The coral microbiome was dominated by Uncultured Betaproteobacteria in the order EC94 (16%), Unknown-Bacteria (15%), family Cenarchaeaceae (12 %), Burkholderiaceae (10%), and Hahellaceae (10%). The constant coral-holobiont composition along the studied depth range suggests that identity of the symbionts is not responsible for the coral’s broad depth range and adaptation to low light environments.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="castro-mejia_extraction_2018" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1007/978-1-4939-8682-8_5" rel="external nofollow noopener">Extraction and Purification of Viruses from Fecal Samples for Metagenome and Morphology Analyses</a></div> <div class="author"> Josué L. Castro-Mejía, Ling Deng, Finn K. Vogensen, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Alejandro Reyes, Dennis S. Nielsen' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>In The Human Virome: Methods and Protocols</em> , Dec 2018 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1007/978-1-4939-8682-8_5" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The human enteric virome consists of endogenous retro elements and viruses that infect the host and members of the gut microbiome (GM). Mounting evidence suggests that the gut virome plays a central role in maintaining homeostasis and via the GM influences immunology of the host. To thoroughly characterize the gut virome, it is often very useful to first separate and concentrate extracellular viral-like particles (eVLPs) enabling an integrative characterization of them. Here, we describe a detailed protocol for extraction and concentration of the viral fraction from fecal samples based on a polyethylene glycol precipitation (PEG) approach. These procedures maximize the yields of eVLPs (and their DNA) with high purity well suited for down-stream analysis such as quantification and morphological assessment, determination of phage-host pairs as well as virome sequencing.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="restrepo_draft_2018" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1128/genomea.00605-18" rel="external nofollow noopener">Draft Genome Sequence of a White Spot Syndrome Virus Isolate Obtained in Ecuador</a></div> <div class="author"> Leda Restrepo, Alejandro Reyes, Leandro Bajaña, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Irma Betancourt, Bonny Bayot' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Genome Announcements</em>, Jun 2018 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1128/genomea.00605-18" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>White spot syndrome virus (WSSV) is the most devastating viral disease affecting cultivated shrimp around the world. Currently, there is no reported genetic information on WSSV affecting Penaeus vannamei in Ecuador. Therefore, we determined the genome of a WSSV isolate from shrimp and compared it with the genomes of isolates from other geographical locations.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="restrepo_pirvp_2018" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/s41598-018-30903-x" rel="external nofollow noopener">PirVP genes causing AHPND identified in a new Vibrio species (Vibrio punensis) within the commensal Orientalis clade</a></div> <div class="author"> Leda Restrepo, Bonny Bayot, Sebastián Arciniegas, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Leandro Bajaña, Irma Betancourt, Fanny Panchana, Alejandro Reyes Muñoz' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Scientific Reports</em>, Aug 2018 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/s41598-018-30903-x" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Acute hepatopancreatic necrosis disease (AHPND) has extended rapidly, causing alarming shrimp mortalities. Initially, the only known causative agent was Vibrio parahaemolyticus carrying a plasmid coding for the mortal toxins PirVP. Recently, it has been found that the plasmid and hence the disease, could be transferred among members of the Harveyi clade. The current study performs a genomic characterization of an isolate capable of developing AHPND in shrimp. Mortality studies and molecular and histopathological analyses showed the infection capacity of the strain. Multilocus sequence analysis placed the bacteria as a member of the Orientalis clade, well known for containing commensal and even probiotic bacteria used in the shrimp industry. Further whole genome comparative analyses, including Vibrio species from the Orientalis clade, and phylogenomic metrics (TETRA, ANI and DDH) showed that the isolate belongs to a previously unidentified species, now named Vibrio punensis sp. nov. strain BA55. Our findings show that the gene transfer capacity of Vibrio species goes beyond the clade classification, demonstrating a new pathogenic capacity to a previously known commensal clade. The presence of these genes in a different Vibrio clade may contribute to the knowledge of the Vibrio pathogenesis and has major implications for the spread of emerging diseases.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="quintanilla_local_2018" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/s41598-018-33007-8" rel="external nofollow noopener">Local confinement of disease-related microbiome facilitates recovery of gorgonian sea fans from necrotic-patch disease</a></div> <div class="author"> Elena Quintanilla, Catalina Ramírez-Portilla, Boahemaa Adu-Oppong, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Gretchen Walljasper, Stefanie P. Glaeser, Thomas Wilke, Alejandro Reyes Muñoz, Juan A. Sánchez' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Scientific Reports</em>, Oct 2018 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/s41598-018-33007-8" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Microbiome disruptions triggering disease outbreaks are increasingly threatening corals worldwide. In the Tropical Eastern Pacific, a necrotic-patch disease affecting gorgonian corals (sea fans, Pacifigorgia spp.) has been observed in recent years. However, the composition of the microbiome and its disease-related disruptions remain unknown in these gorgonian corals. Therefore, we analysed 16S rRNA gene amplicons from tissues of healthy colonies (n = 19) and from symptomatic-asymptomatic tissues of diseased colonies (n = 19) of Pacifigorgia cairnsi (Gorgoniidae: Octocorallia) in order to test for disease-related changes in the bacterial microbiome. We found that potential endosymbionts (mostly Endozoicomonas spp.) dominate the core microbiome in healthy colonies. Moreover, healthy tissues differed in community composition and functional profile from those of the symptomatic tissues but did not show differences to asymptomatic tissues of the diseased colonies. A more diverse set of bacteria was observed in symptomatic tissues, together with the decline in abundance of the potential endosymbionts from the healthy core microbiome. Furthermore, according to a comparative taxonomy-based functional profiling, these symptomatic tissues were characterized by the increase in heterotrophic, ammonia oxidizer and dehalogenating bacteria and by the depletion of nitrite and sulphate reducers. Overall, our results suggest that the bacterial microbiome associated with the disease behaves opportunistically and is likely in a state of microbial dysbiosis. We also conclude that the confinement of the disease-related consortium to symptomatic tissues may facilitate colony recovery.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div id="dutilh_editorial_2017" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fmicb.2017.01710" rel="external nofollow noopener">Editorial: Virus Discovery by Metagenomics: The (Im)possibilities</a></div> <div class="author"> Bas E. Dutilh, Alejandro Reyes, Richard J. Hall, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Katrine L. Whiteson' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Oct 2017 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2017.01710" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div id="borbon-garcia_captivity_2017" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fmicb.2017.01316" rel="external nofollow noopener">Captivity Shapes the Gut Microbiota of Andean Bears: Insights into Health Surveillance</a></div> <div class="author"> Andrea Borbón-García, Alejandro Reyes, Martha Vives-Flórez, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Susana Caballero' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Oct 2017 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2017.01316" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The Andean bear is an endemic species of the tropical Andes who has an almost exclusively plant-based diet. Since herbivorous mammals do not carry enzymes for fiber degradation, the establishment of symbiosis with cellulolytic microorganisms in their gastrointestinal (GI) tract is necessary to help them fulfill their nutritional needs. Furthermore, as described for other mammals, a stable, diverse, and balanced gut microbial composition is an indicator of a healthy status of the host; under disturbances this balance can be lost, leading to potential diseases of the host. The goal of this study was to describe the gut microbiota of wild and captive Andean bears and determine how habitat status influences the composition and diversity of the gut symbiotic community. Fecal samples from wild (n = 28) and captive (n = 8) Andean bears were collected in “Reserva Pantano de Martos” and “Fundación Bioandina”, Colombia. Composition and diversity analyses were performed using amplicons from the V4 region of the 16S rDNA gene sequenced using the Ion PGM platform. PICRUSt algorithm was applied to predict the gene content of the gut microbiome of wild and captive Andean bears. A total of 5,411 and 838 OTUs were identified for wild and captive bears, respectively. Captive bears contained a lower number of bacterial phyla (n = 7) compared to wild individuals (n = 9). Proteobacteria (59.03%) and Firmicutes (14.03%) were the phyla that contributed the most to differences between wild and captive bears (overall dissimilarity = 87.72%). At family level, Enterobacteriaceae drove the main differences between the two groups (13.7%). PICRUSt metagenomics predictions suggested a similar pattern of relative abundance of gene families associated with the metabolism of carbohydrates across samples in wild individuals, despite the taxonomic differences of their gut microbiota. Captivity alters the availability and diversity of food resources, which likely reduces microbiota richness and diversity compared to wild individuals. Further considerations should be taken into account for nutritional schemes improving ex-situ conservation and its potential as a surveillance tool of endangered populations of wild Andean bears.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="rodriguez-castano_advances_2017" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fgene.2016.00224" rel="external nofollow noopener">Advances in Gut Microbiome Research, Opening New Strategies to Cope with a Western Lifestyle</a></div> <div class="author"> Gina P. Rodriguez-Castaño, Alejandro Caro-Quintero, Alejandro Reyes, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Fernando Lizcano' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Frontiers in Genetics</em>, Oct 2017 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fgene.2016.00224" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The “westernization” of global eating and lifestyle habits is associated with the growing rate of chronic diseases, mainly cardiovascular diseases, cancer, type 2 diabetes mellitus, and respiratory diseases. The primary prevention approach is to make nutritional and behavioral changes, however, there is another important determinant of our health that only recently has been considered and is the presence of beneficial microorganisms and their products in our gastrointestinal tract. Microorganisms living in our body can alter the fate of food, drugs, hormones, and xenobiotics, and recent studies point to the use of microorganisms that can counteract the harmful effects of certain compounds introduced or produced endogenously in our body. This review considers the effects of the western lifestyle on adiposity, glucose metabolism, oxidative markers and inflammation profile, emphasizes on the studies that have investigated bacterial strains and products of their metabolism that are beneficial under this lifestyle, and examines the screening strategies that recent studies are using to select the most promising probiotic isolates. In addition, we consider the relevance of studying the microbiota of metabolically healthy people under a western lifestyle for the understanding of the key components that delay the development of chronic diseases.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="reyes_use_2017" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.2147/AGG.S136574" rel="external nofollow noopener">Use of profile hidden Markov models in viral discovery: current insights</a></div> <div class="author"> Alejandro Reyes, João Marcelo P Alves, Alan Mitchell Durham, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Arthur Gruber' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Advances in Genomics and Genetics</em>, Jul 2017 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.2147/AGG.S136574" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Sequence similarity searches are the bioinformatic cornerstone of molecular sequence analysis for all domains of life. However, large amounts of divergence between organisms, such as those seen among viruses, can significantly hamper analyses. Profile hidden Markov models (profile HMMs) are among the most successful approaches for dealing with this problem, which represent an invaluable tool for viral identification efforts. Profile HMMs are statistical models that convert information from a multiple sequence alignment into a set of probability values that reflect position-specific variation levels in all members of evolutionarily related sequences. Since profile HMMs represent a wide spectrum of variation, these models show higher sensitivity than conventional similarity methods such as BLAST for the detection of remote homologs. In recent years, there has been an effort to compile viral sequences from different viral taxonomic groups into integrated databases, such as Prokaryotic Virus Orthlogous Groups (pVOGs) and database of profile HMMs (vFam) database, which provide functional annotation, multiple sequence alignments, and profile HMMs. Since these databases rely on viral sequences collected from GenBank and RefSeq, they suffer in variable extent from uneven taxonomic sampling, with low sequence representation of many viral groups, which affects the efficacy of the models. One of the interesting applications of viral profile HMMs is the detection and sequence reconstruction of specific viral genomes from metagenomic data. In fact, several DNA assembly programs that use profile HMMs as seeds have been developed to identify and build gene-sized assemblies or viral genome sequences of unrestrained length, using conventional and progressive assembly approaches, respectively. In this review, we address these aspects and cover some up-to-date information on viral genomics that should be considered in the choice of molecular markers for viral discovery. Finally, we propose a roadmap for rational development of viral profile HMMs and discuss the main challenges associated with this task.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="buitrago_florez_changing_2017" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3102/0034654317710096" rel="external nofollow noopener">Changing a Generation’s Way of Thinking: Teaching Computational Thinking Through Programming</a></div> <div class="author"> Francisco Buitrago Flórez, Rubby Casallas, Marcela Hernández, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Alejandro Reyes, Silvia Restrepo, Giovanna Danies' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Review of Educational Research</em>, Aug 2017 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3102/0034654317710096" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Computational thinking (CT) uses concepts that are essential to computing and information science to solve problems, design and evaluate complex systems, and understand human reasoning and behavior. This way of thinking has important implications in computer sciences as well as in almost every other field. Therefore, we contend that CT should be taught in elementary schools and included in every university’s educational curriculum. Several studies that measure the impact of teaching programming, analytical thinking, and CT have been conducted. In this review, we analyze and discuss findings from these studies and highlight the importance of learning programming with a focus on the development of CT skills at a young age. We also describe the tools that are available to improve the teaching of CT and provide a state-of-the-art overview of how programming is being taught at schools and universities in Colombia and around the world.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"><li> <div class="row"> <div id="alves_genseed-hmm_2016" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.3389/fmicb.2016.00269" rel="external nofollow noopener">GenSeed-HMM: A Tool for Progressive Assembly Using Profile HMMs as Seeds and its Application in Alpavirinae Viral Discovery from Metagenomic Data</a></div> <div class="author"> João M. P. Alves, André L. Oliveira, Tatiana O. M. Sandberg, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Jaime L. Moreno-Gallego, Marcelo A. F. Toledo, Elisabeth M. M. Moura, Liliane S. Oliveira, Alan M. Durham, Dolores U. Mehnert, Paolo M. de A. Zanotto, Alejandro Reyes, Arthur Gruber' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Aug 2016 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2016.00269" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>This work reports the development of GenSeed-HMM, a program that implements seed-driven progressive assembly, an approach to reconstruct specific sequences from unassembled data, starting from short nucleotide or protein seed sequences or profile Hidden Markov Models (HMM). The program can use any one of a number of sequence assemblers. Assembly is performed in multiple steps and relatively few reads are used in each cycle, consequently the program demands low computational resources. As a proof-of-concept and to demonstrate the power of HMM-driven progressive assemblies, GenSeed-HMM was applied to metagenomic datasets in the search for diverse ssDNA bacteriophages from the recently described Alpavirinae subfamily. Profile HMMs were built using Alpavirinae-specific regions from multiple sequence alignments (MSA) using either the viral protein 1 (VP1; major capsid protein) or VP4 (genome replication initiation protein). These profile HMMs were used by GenSeed-HMM (running Newbler assembler) as seeds to reconstruct viral genomes from sequencing datasets of human fecal samples. All contigs obtained were annotated and taxonomically classified using similarity searches and phylogenetic analyses. The most specific profile HMM seed enabled the reconstruction of 45 partial or complete Alpavirinae genomic sequences. A comparison with conventional (global) assembly of the same original dataset, using Newbler in a standalone execution, revealed that GenSeed-HMM outperformed global genomic assembly in several metrics employed. This approach is capable of detecting organisms that have not been used in the construction of the profile HMM, which opens up the possibility of diagnosing novel viruses, without previous specific information, constituting a de novo diagnosis. Additional applications include, but are not limited to, the specific assembly of extrachromosomal elements such as plastid and mitochondrial genomes from metagenomic data. Profile HMM seeds can also be used to reconstruct specific protein coding genes for gene diversity studies, and to determine all possible gene variants present in a metagenomic sample. Such surveys could be useful to detect the emergence of drug-resistance variants in sensitive environments such as hospitals and animal production facilities, where antibiotics are regularly used. Finally, GenSeed-HMM can be used as an adjunct for gap closure on assembly finishing projects, by using multiple contig ends as anchored seeds.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div id="reyes_gut_2015" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1073/pnas.1514285112" rel="external nofollow noopener">Gut DNA viromes of Malawian twins discordant for severe acute malnutrition</a></div> <div class="author"> Alejandro Reyes, Laura V. Blanton, Song Cao, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Guoyan Zhao, Mark Manary, Indi Trehan, Michelle I. Smith, David Wang, Herbert W. Virgin, Forest Rohwer, Jeffrey I. Gordon' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences</em>, Sep 2015 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1073/pnas.1514285112" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The bacterial component of the human gut microbiota undergoes a definable program of postnatal development. Evidence is accumulating that this program is disrupted in children with severe acute malnutrition (SAM) and that their persistent gut microbiota immaturity, which is not durably repaired with current ready-to-use therapeutic food (RUTF) interventions, is causally related to disease pathogenesis. To further characterize gut microbial community development in healthy versus malnourished infants/children, we performed a time-series metagenomic study of DNA isolated from virus-like particles (VLPs) recovered from fecal samples collected during the first 30 mo of postnatal life from eight pairs of mono- and dizygotic Malawian twins concordant for healthy growth and 12 twin pairs discordant for SAM. Both members of discordant pairs were sampled just before, during, and after treatment with a peanut-based RUTF. Using Random Forests and a dataset of 17,676 viral contigs assembled from shotgun sequencing reads of VLP DNAs, we identified viruses that distinguish different stages in the assembly of the gut microbiota in the concordant healthy twin pairs. This developmental program is impaired in both members of SAM discordant pairs and not repaired with RUTF. Phage plus members of the Anelloviridae and Circoviridae families of eukaryotic viruses discriminate discordant from concordant healthy pairs. These results disclose that apparently healthy cotwins in discordant pairs have viromes associated with, although not necessarily mediators, of SAM; as such, they provide a human model for delineating normal versus perturbed postnatal acquisition and retention of the gut microbiota’s viral component in populations at risk for malnutrition.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div id="seedorf_bacteria_2014" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1016/j.cell.2014.09.008" rel="external nofollow noopener">Bacteria from Diverse Habitats Colonize and Compete in the Mouse Gut</a></div> <div class="author"> Henning Seedorf, Nicholas W. Griffin, Vanessa K. Ridaura, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Alejandro Reyes, Jiye Cheng, Federico E. Rey, Michelle I. Smith, Gabriel M. Simon, Rudolf H. Scheffrahn, Dagmar Woebken, Alfred M. Spormann, William Van Treuren, Luke K. Ursell, Megan Pirrung, Adam Robbins-Pianka, Brandi L. Cantarel, Vincent Lombard, Bernard Henrissat, Rob Knight, Jeffrey I. Gordon' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>Cell</em>, Oct 2014 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2014.09.008" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>To study how microbes establish themselves in a mammalian gut environment, we colonized germ-free mice with microbial communities from human, zebrafish, and termite guts, human skin and tongue, soil, and estuarine microbial mats. Bacteria from these foreign environments colonized and persisted in the mouse gut; their capacity to metabolize dietary and host carbohydrates and bile acids correlated with colonization success. Cohousing mice harboring these xenomicrobiota or a mouse cecal microbiota, along with germ-free “bystanders,” revealed the success of particular bacterial taxa in invading guts with established communities and empty gut habitats. Unanticipated patterns of ecological succession were observed; for example, a soil-derived bacterium dominated even in the presence of bacteria from other gut communities (zebrafish and termite), and human-derived bacteria colonized germ-free bystander mice before mouse-derived organisms. This approach can be generalized to address a variety of mechanistic questions about succession, including succession in the context of microbiota-directed therapeutics.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="dorsett_hcodes_2014" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1016/j.molcel.2014.10.024" rel="external nofollow noopener">HCoDES Reveals Chromosomal DNA End Structures with Single-Nucleotide Resolution</a></div> <div class="author"> Yair Dorsett, Yanjiao Zhou, Anthony T. Tubbs, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Bo-Ruei Chen, Caitlin Purman, Baeck-Seung Lee, Rosmy George, Andrea L. Bredemeyer, Jiang-yang Zhao, Erica Sodergen, George M. Weinstock, Nathan D. Han, Alejandro Reyes, Eugene M. Oltz, Dale Dorsett, Ziva Misulovin, Jacqueline E. Payton, Barry P. Sleckman' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>Molecular Cell</em>, Dec 2014 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1016/j.molcel.2014.10.024" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The structure of broken DNA ends is a critical determinant of the pathway used for DNA double-strand break (DSB) repair. Here, we develop an approach involving the hairpin capture of DNA end structures (HCoDES), which elucidates chromosomal DNA end structures at single-nucleotide resolution. HCoDES defines structures of physiologic DSBs generated by the RAG endonuclease, as well as those generated by nucleases widely used for genome editing. Analysis of G1 phase cells deficient in H2AX or 53BP1 reveals DNA ends that are frequently resected to form long single-stranded overhangs that can be repaired by mutagenic pathways. In addition to 3′ overhangs, many of these DNA ends unexpectedly form long 5′ single-stranded overhangs. The divergence in DNA end structures resolved by HCoDES suggests that H2AX and 53BP1 may have distinct activities in end protection. Thus, the high-resolution end structures obtained by HCoDES identify features of DNA end processing during DSB repair.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"> <li> <div class="row"> <div id="lim_discovery_2013" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1016/j.virol.2012.12.005" rel="external nofollow noopener">Discovery of STL polyomavirus, a polyomavirus of ancestral recombinant origin that encodes a unique T antigen by alternative splicing</a></div> <div class="author"> Efrem S. Lim, Alejandro Reyes, Martin Antonio, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Debasish Saha, Usman N. Ikumapayi, Mitchell Adeyemi, O. Colin Stine, Rebecca Skelton, Daniel C. Brennan, Rajhab S. Mkakosya, Mark J. Manary, Jeffrey I. Gordon, David Wang' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>Virology</em>, Feb 2013 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1016/j.virol.2012.12.005" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The family Polyomaviridae is comprised of circular double-stranded DNA viruses, several of which are associated with diseases, including cancer, in immunocompromised patients. Here we describe a novel polyomavirus recovered from the fecal microbiota of a child in Malawi, provisionally named STL polyomavirus (STLPyV). We detected STLPyV in clinical stool specimens from USA and The Gambia at up to 1% frequency. Complete genome comparisons of two STLPyV strains demonstrated 5.2% nucleotide divergence. Alternative splicing of the STLPyV early region yielded a unique form of T antigen, which we named 229T, in addition to the expected large and small T antigens. STLPyV has a mosaic genome and shares an ancestral recombinant origin with MWPyV. The discovery of STLPyV highlights a novel alternative splicing strategy and advances our understanding of the complex evolutionary history of polyomaviruses.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="reyes_gnotobiotic_2013" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1073/pnas.1319470110" rel="external nofollow noopener">Gnotobiotic mouse model of phage–bacterial host dynamics in the human gut</a></div> <div class="author"> Alejandro Reyes, Meng Wu, Nathan P. McNulty, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Forest L. Rohwer, Jeffrey I. Gordon' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences</em>, Dec 2013 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1073/pnas.1319470110" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Bacterial viruses (phages) are the most abundant biological group on Earth and are more genetically diverse than their bacterial prey/hosts. To characterize their role as agents shaping gut microbial community structure, adult germ-free mice were colonized with a consortium of 15 sequenced human bacterial symbionts, 13 of which harbored one or more predicted prophages. One member, Bacteroides cellulosilyticus WH2, was represented by a library of isogenic transposon mutants that covered 90% of its genes. Once assembled, the community was subjected to a staged phage attack with a pool of live or heat-killed virus-like particles (VLPs) purified from the fecal microbiota of five healthy humans. Shotgun sequencing of DNA from the input pooled VLP preparation plus shotgun sequencing of gut microbiota samples and purified fecal VLPs from the gnotobiotic mice revealed a reproducible nonsimultaneous pattern of attack extending over a 25-d period that involved five phages, none described previously. This system allowed us to (i) correlate increases in specific phages present in the pooled VLPs with reductions in the representation of particular bacterial taxa, (ii) provide evidence that phage resistance occurred because of ecological or epigenetic factors, (iii) track the origin of each of the five phages among the five human donors plus the extent of their genome variation between and within recipient mice, and (iv) establish the dramatic in vivo fitness advantage that a locus within a B. cellulosilyticus prophage confers upon its host. Together, these results provide a defined community-wide view of phage–bacterial host dynamics in the gut.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2012</h2> <ol class="bibliography"> <li> <div class="row"> <div id="cardona_genomic_2012" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.5772/31276" rel="external nofollow noopener">Genomic Variability of Mycobacterium tuberculosis</a></div> <div class="author"> Mara Mercedes, Ginna Hernndez-Neuta, Ivn Hernndez-Neuta, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Andrea Sandoval, Andrs Cubillos-Ruiz, Alejandro Reyes, Patricia Del' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>In Understanding Tuberculosis - Deciphering the Secret Life of the Bacilli</em> , Feb 2012 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.5772/31276" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div id="kim_noncomplementation_2012" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1534/g3.112.002550" rel="external nofollow noopener">A Noncomplementation Screen for Quantitative Trait Alleles in Saccharomyces cerevisiae</a></div> <div class="author"> Hyun Seok Kim, Juyoung Huh, Linda Riles, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Alejandro Reyes, Justin C Fay' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>G3 Genes\textbarGenomes\textbarGenetics</em>, Jul 2012 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1534/g3.112.002550" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Both linkage and linkage disequilibrium mapping provide well-defined approaches to mapping quantitative trait alleles. However, alleles of small effect are particularly difficult to refine to individual genes and causative mutations. Quantitative noncomplementation provides a means of directly testing individual genes for quantitative trait alleles in a fixed genetic background. Here, we implement a genome-wide noncomplementation screen for quantitative trait alleles that affect colony color or size by using the yeast deletion collection. As proof of principle, we find a previously known allele of CYS4 that affects colony color and a novel allele of CTT1 that affects resistance to hydrogen peroxide. To screen nearly 4700 genes in nine diverse yeast strains, we developed a high-throughput robotic plating assay to quantify colony color and size. Although we found hundreds of candidate alleles, reciprocal hemizygosity analysis of a select subset revealed that many of the candidates were false positives, in part the result of background-dependent haploinsufficiency or second-site mutations within the yeast deletion collection. Our results highlight the difficulty of identifying small-effect alleles but support the use of noncomplementation as a rapid means of identifying quantitative trait alleles of large effect.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="forsberg_shared_2012" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1126/science.1220761" rel="external nofollow noopener">The Shared Antibiotic Resistome of Soil Bacteria and Human Pathogens</a></div> <div class="author"> Kevin J. Forsberg, Alejandro Reyes, Bin Wang, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Elizabeth M. Selleck, Morten O. A. Sommer, Gautam Dantas' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Science</em>, Aug 2012 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1126/science.1220761" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Soil microbiota represent one of the ancient evolutionary origins of antibiotic resistance and have been proposed as a reservoir of resistance genes available for exchange with clinical pathogens. Using a high-throughput functional metagenomic approach in conjunction with a pipeline for the de novo assembly of short-read sequence data from functional selections (termed PARFuMS), we provide evidence for recent exchange of antibiotic resistance genes between environmental bacteria and clinical pathogens. We describe multidrug-resistant soil bacteria containing resistance cassettes against five classes of antibiotics (β-lactams, aminoglycosides, amphenicols, sulfonamides, and tetracyclines) that have perfect nucleotide identity to genes from diverse human pathogens. This identity encompasses noncoding regions as well as multiple mobilization sequences, offering not only evidence of lateral exchange but also a mechanism by which antibiotic resistance disseminates.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="reyes_going_2012" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/nrmicro2853" rel="external nofollow noopener">Going viral: next-generation sequencing applied to phage populations in the human gut</a></div> <div class="author"> Alejandro Reyes, Nicholas P. Semenkovich, Katrine Whiteson, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Forest Rohwer, Jeffrey I. Gordon' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Nature Reviews Microbiology</em>, Sep 2012 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/nrmicro2853" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Methods for purifying virus-like particles (VLPs) from microbial communities, procedures for amplifying the small quantities of DNA that are recovered from VLPs, advances in next-generation sequencing, and a number of new computational approaches have laid the foundations for a ’new age of phage’, in which rapid progress is being made in characterizing the viral diversity and virus–bacterial host dynamics in the microbial communities residing in a broad range of habitats, including those associated with our human bodies.Phage genomes have limited sequence conservation, making comparative genomics difficult. There is no conserved phylogenetic marker. Despite these obstacles, new techniques have been developed to classify phages.The phage community in the human gut is complex but appears to be much more stable than those in other habitats, such as the ocean. Patterns of temporal and functional variation are being defined using metagenomics. Gnotobiotic animal models hold promise for further characterization of the role of phages in shaping the properties of the human gut microbiota, including its responses to various perturbations.New insights about the microbial ecology in humans have been gleaned from comparative metagenomics studies and have rekindled an interest in phage therapy. Therapeutic goals may include enhancing the ability of probiotic consortia to establish themselves, and the addition of novel functions to the gut microbiome. Representative preclinical models are needed for proof-of-principle, proof-of-efficacy, dosing and safety tests.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="siebrasse_identification_2012" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1128/jvi.01210-12" rel="external nofollow noopener">Identification of MW Polyomavirus, a Novel Polyomavirus in Human Stool</a></div> <div class="author"> Erica A. Siebrasse, Alejandro Reyes, Efrem S. Lim, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Guoyan Zhao, Rajhab S. Mkakosya, Mark J. Manary, Jeffrey I. Gordon, David Wang' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Journal of Virology</em>, Oct 2012 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1128/jvi.01210-12" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We have discovered a novel polyomavirus present in multiple human stool samples. The virus was initially identified by shotgun pyrosequencing of DNA purified from virus-like particles isolated from a stool sample collected from a healthy child from Malawi. We subsequently sequenced the virus’ 4,927-bp genome, which has been provisionally named MW polyomavirus (MWPyV). The virus has genomic features characteristic of the family Polyomaviridae but is highly divergent from other members of this family. It is predicted to encode the large T antigen and small T antigen early proteins and the VP1, VP2, and VP3 structural proteins. A real-time PCR assay was designed and used to screen 514 stool samples from children with diarrhea in St. Louis, MO; 12 specimens were positive for MWPyV. Comparison of the whole-genome sequences of the index Malawi case and one St. Louis case demonstrated that the two strains of MWPyV varied by 5.3% at the nucleotide level. The number of polyomaviruses found in the human body continues to grow, raising the question of how many more species have yet to be identified and what roles they play in humans with and without manifest disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="reyes_is-seq_2012" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1186/1471-2164-13-249" rel="external nofollow noopener">IS-seq: a novel high throughput survey of in vivo IS6110 transposition in multiple Mycobacterium tuberculosis genomes</a></div> <div class="author"> Alejandro Reyes, Andrea Sandoval, Andrés Cubillos-Ruiz, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Katherine E. Varley, Ivan Hernández-Neuta, Sofía Samper, Carlos Martín, María Jesús García, Viviana Ritacco, Lucelly López, Jaime Robledo, María Mercedes Zambrano, Robi D. Mitra, Patricia Del Portillo' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>BMC Genomics</em>, Jun 2012 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1186/1471-2164-13-249" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The insertion element IS6110 is one of the main sources of genomic variability in Mycobacterium tuberculosis, the etiological agent of human tuberculosis. Although IS 6110 has been used extensively as an epidemiological marker, the identification of the precise chromosomal insertion sites has been limited by technical challenges. Here, we present IS-seq, a novel method that combines high-throughput sequencing using Illumina technology with efficient combinatorial sample multiplexing to simultaneously probe 519 clinical isolates, identifying almost all the flanking regions of the element in a single experiment.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2011</h2> <ol class="bibliography"><li> <div class="row"> <div id="goodman_extensive_2011" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1073/pnas.1102938108" rel="external nofollow noopener">Extensive personal human gut microbiota culture collections characterized and manipulated in gnotobiotic mice</a></div> <div class="author"> Andrew L. Goodman, George Kallstrom, Jeremiah J. Faith, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Alejandro Reyes, Aimee Moore, Gautam Dantas, Jeffrey I. Gordon' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences</em>, Apr 2011 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1073/pnas.1102938108" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The proportion of the human gut bacterial community that is recalcitrant to culture remains poorly defined. In this report, we combine high-throughput anaerobic culturing techniques with gnotobiotic animal husbandry and metagenomics to show that the human fecal microbiota consists largely of taxa and predicted functions that are represented in its readily cultured members. When transplanted into gnotobiotic mice, complete and cultured communities exhibit similar colonization dynamics, biogeographical distribution, and responses to dietary perturbations. Moreover, gnotobiotic mice can be used to shape these personalized culture collections to enrich for taxa suited to specific diets. We also demonstrate that thousands of isolates from a single donor can be clonally archived and taxonomically mapped in multiwell format to create personalized microbiota collections. Retrieving components of a microbiota that have coexisted in single donors who have physiologic or disease phenotypes of interest and reuniting them in various combinations in gnotobiotic mice should facilitate preclinical studies designed to determine the degree to which tractable bacterial taxa are able to transmit donor traits or influence host biology.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2010</h2> <ol class="bibliography"><li> <div class="row"> <div id="reyes_viruses_2010" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1038/nature09199" rel="external nofollow noopener">Viruses in the faecal microbiota of monozygotic twins and their mothers</a></div> <div class="author"> Alejandro Reyes, Matthew Haynes, Nicole Hanson, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Florent E. Angly, Andrew C. Heath, Forest Rohwer, Jeffrey I. Gordon' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Jul 2010 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1038/nature09199" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Viral diversity and life cycles are poorly understood in the human gut and other body habitats. Phages and their encoded functions may provide informative signatures of a human microbiota and of microbial community responses to various disturbances, and may indicate whether community health or dysfunction is manifest after apparent recovery from a disease or therapeutic intervention. Here we report sequencing of the viromes (metagenomes) of virus-like particles isolated from faecal samples collected from healthy adult female monozygotic twins and their mothers at three time points over a one-year period. We compared these data sets with data sets of sequenced bacterial 16S ribosomal RNA genes and total-faecal-community DNA. Co-twins and their mothers share a significantly greater degree of similarity in their faecal bacterial communities than do unrelated individuals. In contrast, viromes are unique to individuals regardless of their degree of genetic relatedness. Despite remarkable interpersonal variations in viromes and their encoded functions, intrapersonal diversity is very low, with \textgreater95% of virotypes retained over the period surveyed, and with viromes dominated by a few temperate phages that exhibit remarkable genetic stability. These results indicate that a predatory viral–microbial dynamic, manifest in a number of other characterized environmental ecosystems, is notably absent in the very distal intestine.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2009</h2> <ol class="bibliography"><li> <div class="row"> <div id="garcia_differential_2009" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1016/j.aquaculture.2009.01.020" rel="external nofollow noopener">Differential gene expression in White Spot Syndrome Virus (WSSV)-infected naïve and previously challenged Pacific white shrimp Penaeus (Litopenaeus) vannamei</a></div> <div class="author"> Juan C. García, Alejandro Reyes, Marcela Salazar, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Clarissa B. Granja' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Aquaculture</em>, Apr 2009 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1016/j.aquaculture.2009.01.020" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>White Spot Syndrome Virus (WSSV) is the most devastating viral disease affecting cultivated shrimp around the world. The CENIACUA breeding programme included host resistance as strategy to control WSSV. Since 1999, all families were subjected to WSSV challenge tests and up to 5% of the animals survived. We suspect that a unique genetic profile could be responsible for survival. By Subtractive Suppression Hybridization improved with Mirror Orientation Selection we compared gene expression in haemocytes of WSSV-infected shrimp; seven were previously challenged (pre-challenged) and 10 naïve. Out of 1167 clones, 993 (85%) were obtained from the WSSV-infected pre-challenged library, while 174 (15%) were from the WSSV-infected naïve library. Differential expression was confirmed by hybridization using PCR DIG-labelled probes from unsubtracted cDNA. The 216 (18.5%) clones differentially expressed in the WSSV-infected pre-challenged library and 23 (2%) in the WSSV-infected naïve animals were sequenced. Relative expression of relevant gene products from both libraries was confirmed by reverse transcription real time PCR. Genes related to shrimp immune response, such as the antimicrobial peptide penaeidin, crustin, C-type lectin, protease inhibitor and chitin binding domain-containing protein were found in the library of pre-challenged shrimp. On the other hand, genes from the naïve library coded for viral structural and functional proteins. To our knowledge, this is the first report demonstrating that penaeidin-3 isoforms and crustin are over expressed in haemocytes of WSSV-infected pre-challenged P. vannamei.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2007</h2> <ol class="bibliography"> <li> <div class="row"> <div id="portillo_chapter_2007" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/" rel="external nofollow noopener">Chapter 4: Genomics and Proteomics</a></div> <div class="author"> Patricia Del Portillo, Alejandro Reyes, Leiria Salazar, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'María Jesús García' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>In Tuberculosis Textbook</em> , Apr 2007 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div id="reyes_temperature_2007" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1016/j.dci.2006.05.003" rel="external nofollow noopener">Temperature modifies gene expression in subcuticular epithelial cells of white spot syndrome virus-infected Litopenaeus vannamei</a></div> <div class="author"> Alejandro Reyes, Marcela Salazar, and Clarissa Granja </div> <div class="periodical"> <em>Developmental &amp; Comparative Immunology</em>, Jan 2007 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1016/j.dci.2006.05.003" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Subtractive suppressive hybridization was used to identify differentially expressed genes in subcuticular tissues from white spot syndrome virus(WSSV)-infected shrimp kept at different temperatures. Subtractive libraries I and II contained genes expressed at 26 and 33°C, respectively. Three hundred and seventy-nine insert positive clones were selected to confirm differential expression by dot-blot hybridization. Twenty-two clones from library I and eight from library II were sequenced. All sequences from Library I corresponded to white spot syndrome virus genes. From library II, five clones were homologous with previously reported expressed sequence tags of Litopenaeus vannamei, two had similarity with β-actin and one transcript represented an unknown gene. Over-expression of VP15 in shrimp at 26°C was further confirmed by real-time polymerase chain reaction (PCR), whereas β-actin expression was similar in animals kept at both temperatures. Together, our results show that hyperthermia reduces the expression of WSSV genes on shrimp subcuticular epithelial cells.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="olano_mutations_2007" class="col-sm-10"> <div class="title"><a target="_blank" style="font-size: 115%;" href="http://doi.org/10.1016/j.tube.2007.05.011" rel="external nofollow noopener">Mutations in DNA repair genes are associated with the Haarlem lineage of Mycobacterium tuberculosis independently of their antibiotic resistance</a></div> <div class="author"> Juanita Olano, Beatriz López, Alejandro Reyes, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'María Pilar Lemos, Nidia Correa, Patricia Del Portillo, Lucia Barrera, Jaime Robledo, Viviana Ritacco, María Mercedes Zambrano' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Tuberculosis</em>, Nov 2007 </div> <span style="display:inline;" class="__dimensions_badge_embed__" data-doi="10.1016/j.tube.2007.05.011" data-style="small_rectangle"></span><script async="" src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The analysis of the DNA repair genes ogt and ung was carried out in 117 Mycobacterium tuberculosis clinical isolates from Argentina and Colombia in order to explore correlation between mutations in these genes and multi-drug resistance. With the exception of two Beijing family isolates, the rest of the strains harbored either two wild-type or two mutant alleles with identical single nucleotide polymorphisms (SNPs) in each gene (ogt44 and ung501). These ogt44 and ung501 mutations were not associated with multi-drug resistance and occurred simultaneously in circulating Haarlem genotype M. tuberculosis strains. We therefore propose the use of these markers as tools in phylogenetic and epidemiologic studies.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="sticky-bottom mt-5" role="contentinfo"> <div class="container"> © Copyright 2024 BCEM @ Uniandes. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/george-gca/multi-language-al-folio" rel="external nofollow noopener" target="_blank">multi-language-al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?06cae41083477f121be8cd9797ad8e2f"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.min.js" integrity="sha256-rjmgmaB99riUNcdlrDtcAiwtLIojSxNyUFdl+Qh+rB4=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> <script type="module" src="/assets/js/search/ninja-keys.min.js?601a2d3465e2a52bec38b600518d5f70"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"> <div class="modal-footer" slot="footer"> <span class="help"> <svg version="1.0" class="ninja-examplekey" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 1280 1280"> <path d="M1013 376c0 73.4-.4 113.3-1.1 120.2a159.9 159.9 0 0 1-90.2 127.3c-20 9.6-36.7 14-59.2 15.5-7.1.5-121.9.9-255 1h-242l95.5-95.5 95.5-95.5-38.3-38.2-38.2-38.3-160 160c-88 88-160 160.4-160 161 0 .6 72 73 160 161l160 160 38.2-38.3 38.3-38.2-95.5-95.5-95.5-95.5h251.1c252.9 0 259.8-.1 281.4-3.6 72.1-11.8 136.9-54.1 178.5-116.4 8.6-12.9 22.6-40.5 28-55.4 4.4-12 10.7-36.1 13.1-50.6 1.6-9.6 1.8-21 2.1-132.8l.4-122.2H1013v110z"></path> </svg> to select </span> <span class="help"> <svg xmlns="http://www.w3.org/2000/svg" class="ninja-examplekey" viewbox="0 0 24 24"> <path d="M0 0h24v24H0V0z" fill="none"></path> <path d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"></path> </svg> <svg xmlns="http://www.w3.org/2000/svg" class="ninja-examplekey" viewbox="0 0 24 24"> <path d="M0 0h24v24H0V0z" fill="none"></path> <path d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"></path> </svg> to navigate </span> <span class="help"> <span class="ninja-examplekey esc">esc</span> to close </span> <span class="help"> <svg xmlns="http://www.w3.org/2000/svg" class="ninja-examplekey backspace" viewbox="0 0 20 20" fill="currentColor"> <path fill-rule="evenodd" d="M6.707 4.879A3 3 0 018.828 4H15a3 3 0 013 3v6a3 3 0 01-3 3H8.828a3 3 0 01-2.12-.879l-4.415-4.414a1 1 0 010-1.414l4.414-4.414zm4 2.414a1 1 0 00-1.414 1.414L10.586 10l-1.293 1.293a1 1 0 101.414 1.414L12 11.414l1.293 1.293a1 1 0 001.414-1.414L13.414 10l1.293-1.293a1 1 0 00-1.414-1.414L12 8.586l-1.293-1.293z" clip-rule="evenodd"></path> </svg> move to parent </span> </div> </ninja-keys> <script>let searchTheme=determineComputedTheme();const ninjaKeys=document.querySelector("ninja-keys");"dark"===searchTheme?ninjaKeys.classList.add("dark"):ninjaKeys.classList.remove("dark");const openSearchModal=()=>{const e=$("#navbarNav");e.hasClass("show")&&e.collapse("hide"),ninjaKeys.open()};</script> <script>const ninja=document.querySelector("ninja-keys");ninja.data=[{id:"nav-home",title:"Home",section:"Navigation menu",handler:()=>{window.location.href="/"}},{id:"nav-team",title:"Team",description:"",section:"Navigation menu",handler:()=>{window.location.href="/team/"}},{id:"nav-research",title:"Research",description:"",section:"Navigation menu",handler:()=>{window.location.href="/research/"}},{id:"nav-publications",title:"Publications",description:"",section:"Navigation menu",handler:()=>{window.location.href="/publications/"}},{id:"nav-repositories",title:"Repositories",description:"",section:"Navigation menu",handler:()=>{window.location.href="/repositories/"}},{id:"nav-contact",title:"Contact",description:"",section:"Navigation menu",handler:()=>{window.location.href="/contact/"}},{id:"dropdown-bcem-along-the-years",title:"BCEM along the years",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-gallery",title:"Gallery",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"post-google-gemini-updates-flash-1-5-gemma-2-and-project-astra",title:"Google Gemini updates: Flash 1.5, Gemma 2 and Project Astra",description:"We\u2019re sharing updates across our Gemini family of models and a glimpse of Project Astra, our vision for the future of AI assistants.",section:"Posts",handler:()=>{window.open("https://blog.google/technology/ai/google-gemini-update-flash-ai-assistant-io-2024/","_blank")}},{id:"post-a-post-with-tabs",title:"a post with tabs",description:"this is what included tabs in a post could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/tabs/"}},{id:"post-a-post-with-typograms",title:"a post with typograms",description:"this is what included typograms code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/typograms/"}},{id:"post-a-post-that-can-be-cited",title:"a post that can be cited",description:"this is what a post that can be cited looks like",section:"Posts",handler:()=>{window.location.href="/blog/2024/post-citation/"}},{id:"post-a-post-with-pseudo-code",title:"a post with pseudo code",description:"this is what included pseudo code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/pseudocode/"}},{id:"post-a-post-with-code-diff",title:"a post with code diff",description:"this is how you can display code diffs",section:"Posts",handler:()=>{window.location.href="/blog/2024/code-diff/"}},{id:"post-a-post-with-advanced-image-components",title:"a post with advanced image components",description:"this is what advanced image components could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/advanced-images/"}},{id:"post-a-post-with-vega-lite",title:"a post with vega lite",description:"this is what included vega lite code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/vega-lite/"}},{id:"post-a-post-with-geojson",title:"a post with geojson",description:"this is what included geojson code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/geojson-map/"}},{id:"post-a-post-with-echarts",title:"a post with echarts",description:"this is what included echarts code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/echarts/"}},{id:"post-a-post-with-chart-js",title:"a post with chart.js",description:"this is what included chart.js code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/chartjs/"}},{id:"post-a-post-with-tikzjax",title:"a post with TikZJax",description:"this is what included TikZ code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2023/tikzjax/"}},{id:"post-a-post-with-bibliography",title:"a post with bibliography",description:"an example of a blog post with bibliography",section:"Posts",handler:()=>{window.location.href="/blog/2023/post-bibliography/"}},{id:"post-a-post-with-jupyter-notebook",title:"a post with jupyter notebook",description:"an example of a blog post with jupyter notebook",section:"Posts",handler:()=>{window.location.href="/blog/2023/jupyter-notebook/"}},{id:"post-a-post-with-custom-blockquotes",title:"a post with custom blockquotes",description:"an example of a blog post with custom blockquotes",section:"Posts",handler:()=>{window.location.href="/blog/2023/custom-blockquotes/"}},{id:"post-a-post-with-table-of-contents-on-a-sidebar",title:"a post with table of contents on a sidebar",description:"an example of a blog post with table of contents on a sidebar",section:"Posts",handler:()=>{window.location.href="/blog/2023/sidebar-table-of-contents/"}},{id:"post-a-post-with-audios",title:"a post with audios",description:"this is what included audios could look like",section:"Posts",handler:()=>{window.location.href="/blog/2023/audios/"}},{id:"post-a-post-with-videos",title:"a post with videos",description:"this is what included videos could look like",section:"Posts",handler:()=>{window.location.href="/blog/2023/videos/"}},{id:"post-displaying-beautiful-tables-with-bootstrap-tables",title:"displaying beautiful tables with Bootstrap Tables",description:"an example of how to use Bootstrap Tables",section:"Posts",handler:()=>{window.location.href="/blog/2023/tables/"}},{id:"post-a-post-with-table-of-contents",title:"a post with table of contents",description:"an example of a blog post with table of contents",section:"Posts",handler:()=>{window.location.href="/blog/2023/table-of-contents/"}},{id:"post-a-post-with-giscus-comments",title:"a post with giscus comments",description:"an example of a blog post with giscus comments",section:"Posts",handler:()=>{window.location.href="/blog/2022/giscus-comments/"}},{id:"post-displaying-external-posts-on-your-al-folio-blog",title:"Displaying External Posts on Your al-folio Blog",description:"",section:"Posts",handler:()=>{window.open("https://medium.com/@al-folio/displaying-external-posts-on-your-al-folio-blog-b60a1d241a0a?source=rss-17feae71c3c4------2","_blank")}},{id:"post-a-post-with-redirect",title:"a post with redirect",description:"you can also redirect to assets like pdf",section:"Posts",handler:()=>{window.location.href="/assets/pdf/en-us/example_pdf.pdf"}},{id:"post-a-post-with-diagrams",title:"a post with diagrams",description:"an example of a blog post with diagrams",section:"Posts",handler:()=>{window.location.href="/blog/2021/diagrams/"}},{id:"post-a-post-with-github-metadata",title:"a post with github metadata",description:"a quick run down on accessing github metadata.",section:"Posts",handler:()=>{window.location.href="/blog/2020/github-metadata/"}},{id:"post-a-post-with-twitter",title:"a post with twitter",description:"an example of a blog post with twitter",section:"Posts",handler:()=>{window.location.href="/blog/2020/twitter/"}},{id:"post-a-distill-style-blog-post",title:"a distill-style blog post",description:"an example of a distill-style blog post and main elements",section:"Posts",handler:()=>{window.location.href="/blog/2018/distill/"}},{id:"post-a-post-with-disqus-comments",title:"a post with disqus comments",description:"an example of a blog post with disqus comments",section:"Posts",handler:()=>{window.location.href="/blog/2015/disqus-comments/"}},{id:"post-a-post-with-math",title:"a post with math",description:"an example of a blog post with some math",section:"Posts",handler:()=>{window.location.href="/blog/2015/math/"}},{id:"post-a-post-with-code",title:"a post with code",description:"an example of a blog post with some code",section:"Posts",handler:()=>{window.location.href="/blog/2015/code/"}},{id:"post-a-post-with-images",title:"a post with images",description:"this is what included images could look like",section:"Posts",handler:()=>{window.location.href="/blog/2015/images/"}},{id:"post-a-post-with-formatting-and-links",title:"a post with formatting and links",description:"march & april, looking forward to summer",section:"Posts",handler:()=>{window.location.href="/blog/2015/formatting-and-links/"}},{id:"news-a-simple-inline-announcement-with-markdown-emoji-sparkles-smile",title:'A simple inline announcement with Markdown emoji! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20">',description:"",section:"News"},{id:"news-a-long-announcement-with-details",title:"A long announcement with details",description:"",section:"News",handler:()=>{window.location.href="/news/announcement_2/"}},{id:"news-a-simple-inline-announcement",title:"A simple inline announcement.",description:"",section:"News"},{id:"projects-project-7",title:"project 7",description:"with background image",section:"Projects",handler:()=>{window.location.href="/projects/7_project/"}},{id:"projects-project-8",title:"project 8",description:"an other project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/8_project/"}},{id:"projects-project-9",title:"project 9",description:"another project with an image \ud83c\udf89",section:"Projects",handler:()=>{window.location.href="/projects/9_project/"}},{id:"projects-project-1",title:"project 1",description:"with background image",section:"Projects",handler:()=>{window.location.href="/projects/1_project/"}},{id:"projects-project-2",title:"project 2",description:"a project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/2_project/"}},{id:"projects-project-3-with-very-long-name",title:"project 3 with very long name",description:"a project that redirects to another website",section:"Projects",handler:()=>{window.location.href="/projects/3_project/"}},{id:"projects-project-4",title:"project 4",description:"another without an image",section:"Projects",handler:()=>{window.location.href="/projects/4_project/"}},{id:"projects-project-5",title:"project 5",description:"a project with a background image",section:"Projects",handler:()=>{window.location.href="/projects/5_project/"}},{id:"projects-project-6",title:"project 6",description:"a project with no image",section:"Projects",handler:()=>{window.location.href="/projects/6_project/"}},{id:"team-alejandro-castellanos",title:"Alejandro_castellanos",description:"",section:"",handler:()=>{window.location.href="/team/alejandro_castellanos/"}},{id:"team-alejandro-reyes",title:"Alejandro_reyes",description:"",section:"",handler:()=>{window.location.href="/team/alejandro_reyes/"}},{id:"team-andrea-borbon",title:"Andrea_borbon",description:"",section:"",handler:()=>{window.location.href="/team/andrea_borbon/"}},{id:"team-angela-garcia",title:"Angela_garcia",description:"",section:"",handler:()=>{window.location.href="/team/angela_garcia/"}},{id:"team-angela-pena",title:"Angela_pena",description:"",section:"",handler:()=>{window.location.href="/team/angela_pena/"}},{id:"team-aurelia-ayala",title:"Aurelia_ayala",description:"",section:"",handler:()=>{window.location.href="/team/aurelia_ayala/"}},{id:"team-camila-martinez",title:"Camila_martinez",description:"",section:"",handler:()=>{window.location.href="/team/camila_martinez/"}},{id:"team-camilo-garcia",title:"Camilo_garcia",description:"",section:"",handler:()=>{window.location.href="/team/camilo_garcia/"}},{id:"team-carlos-moreno",title:"Carlos_moreno",description:"",section:"",handler:()=>{window.location.href="/team/carlos_moreno/"}},{id:"team-carlos-posada",title:"Carlos_posada",description:"",section:"",handler:()=>{window.location.href="/team/carlos_posada/"}},{id:"team-catalina-ramirez",title:"Catalina_ramirez",description:"",section:"",handler:()=>{window.location.href="/team/catalina_ramirez/"}},{id:"team-daniela-betancurt",title:"Daniela_betancurt",description:"",section:"",handler:()=>{window.location.href="/team/daniela_betancurt/"}},{id:"team-david-alzate",title:"David_alzate",description:"",section:"",handler:()=>{window.location.href="/team/david_alzate/"}},{id:"team-david-botero",title:"David_botero",description:"",section:"",handler:()=>{window.location.href="/team/david_botero/"}},{id:"team-dayanne-chaparro",title:"Dayanne_chaparro",description:"",section:"",handler:()=>{window.location.href="/team/dayanne_chaparro/"}},{id:"team-felipe-sierra",title:"Felipe_sierra",description:"",section:"",handler:()=>{window.location.href="/team/felipe_sierra/"}},{id:"team-gamaliel-lopez",title:"Gamaliel_lopez",description:"",section:"",handler:()=>{window.location.href="/team/gamaliel_lopez/"}},{id:"team-guillermo-luque",title:"Guillermo_luque",description:"",section:"",handler:()=>{window.location.href="/team/guillermo_luque/"}},{id:"team-guillermo-rangel",title:"Guillermo_rangel",description:"",section:"",handler:()=>{window.location.href="/team/guillermo_rangel/"}},{id:"team-irene-blanco",title:"Irene_blanco",description:"",section:"",handler:()=>{window.location.href="/team/irene_blanco/"}},{id:"team-jorge-diaz",title:"Jorge_diaz",description:"",section:"",handler:()=>{window.location.href="/team/jorge_diaz/"}},{id:"team-joshua-kirsch",title:"Joshua_kirsch",description:"",section:"",handler:()=>{window.location.href="/team/joshua_kirsch/"}},{id:"team-juan-anzola",title:"Juan_anzola",description:"",section:"",handler:()=>{window.location.href="/team/juan_anzola/"}},{id:"team-juan-pablo-londono",title:"Juan_pablo_londono",description:"",section:"",handler:()=>{window.location.href="/team/juan_pablo_londono/"}},{id:"team-juan-sebastian-andrade",title:"Juan_sebastian_andrade",description:"",section:"",handler:()=>{window.location.href="/team/juan_sebastian_andrade/"}},{id:"team-juanita-puentes",title:"Juanita_puentes",description:"",section:"",handler:()=>{window.location.href="/team/juanita_puentes/"}},{id:"team-laura-avellaneda",title:"Laura_avellaneda",description:"",section:"",handler:()=>{window.location.href="/team/laura_avellaneda/"}},{id:"team-laura-camelo",title:"Laura_camelo",description:"",section:"",handler:()=>{window.location.href="/team/laura_camelo/"}},{id:"team-laura-forero",title:"Laura_forero",description:"",section:"",handler:()=>{window.location.href="/team/laura_forero/"}},{id:"team-laura-gonzalez",title:"Laura_gonzalez",description:"",section:"",handler:()=>{window.location.href="/team/laura_gonzalez/"}},{id:"team-leda-restrepo",title:"Leda_restrepo",description:"",section:"",handler:()=>{window.location.href="/team/leda_restrepo/"}},{id:"team-leonardo-moreno",title:"Leonardo_moreno",description:"",section:"",handler:()=>{window.location.href="/team/leonardo_moreno/"}},{id:"team-liliane-santana",title:"Liliane_santana",description:"",section:"",handler:()=>{window.location.href="/team/liliane_santana/"}},{id:"team-lorena-monserrate",title:"Lorena_monserrate",description:"",section:"",handler:()=>{window.location.href="/team/lorena_monserrate/"}},{id:"team-luis-chica",title:"Luis_chica",description:"",section:"",handler:()=>{window.location.href="/team/luis_chica/"}},{id:"team-luisa-arias",title:"Luisa_arias",description:"",section:"",handler:()=>{window.location.href="/team/luisa_arias/"}},{id:"team-luisa-matiz",title:"Luisa_matiz",description:"",section:"",handler:()=>{window.location.href="/team/luisa_matiz/"}},{id:"team-luisa-sacristan",title:"Luisa_sacristan",description:"",section:"",handler:()=>{window.location.href="/team/luisa_sacristan/"}},{id:"team-maria-alejandra-ulloa",title:"Maria_alejandra_ulloa",description:"",section:"",handler:()=>{window.location.href="/team/maria_alejandra_ulloa/"}},{id:"team-maria-anotnia-madrid",title:"Maria_anotnia_madrid",description:"",section:"",handler:()=>{window.location.href="/team/maria_anotnia_madrid/"}},{id:"team-maria-fernanda-pena",title:"Maria_fernanda_pena",description:"",section:"",handler:()=>{window.location.href="/team/maria_fernanda_pena/"}},{id:"team-maryam-chaib-demares",title:"Maryam_chaib_demares",description:"",section:"",handler:()=>{window.location.href="/team/maryam_chaib_demares/"}},{id:"team-mauricio-pacheco",title:"Mauricio_pacheco",description:"",section:"",handler:()=>{window.location.href="/team/mauricio_pacheco/"}},{id:"team-mauricio-rico",title:"Mauricio_rico",description:"",section:"",handler:()=>{window.location.href="/team/mauricio_rico/"}},{id:"team-member-template",title:"Member_template",description:"",section:"",handler:()=>{window.location.href="/team/member_template/"}},{id:"team-miguel-parra",title:"Miguel_parra",description:"",section:"",handler:()=>{window.location.href="/team/miguel_parra/"}},{id:"team-nathalia-portilla",title:"Nathalia_portilla",description:"",section:"",handler:()=>{window.location.href="/team/nathalia_portilla/"}},{id:"team-nickole-villabona",title:"Nickole_villabona",description:"",section:"",handler:()=>{window.location.href="/team/nickole_villabona/"}},{id:"team-pablo-cardenas",title:"Pablo_cardenas",description:"",section:"",handler:()=>{window.location.href="/team/pablo_cardenas/"}},{id:"team-ruth-hernandez",title:"Ruth_hernandez",description:"",section:"",handler:()=>{window.location.href="/team/ruth_hernandez/"}},{id:"team-santiago-melo",title:"Santiago_melo",description:"",section:"",handler:()=>{window.location.href="/team/santiago_melo/"}},{id:"team-silvia-rodriguez",title:"Silvia_rodriguez",description:"",section:"",handler:()=>{window.location.href="/team/silvia_rodriguez/"}},{id:"team-zaide-montes",title:"Zaide_montes",description:"",section:"",handler:()=>{window.location.href="/team/zaide_montes/"}},{id:"socials-github",title:"GitHub",section:"Socials",handler:()=>{window.open("https://github.com/BCEM-UniAndes","_blank")}},{id:"socials-x",title:"X",description:"Twitter",section:"Socials",handler:()=>{window.open("https://twitter.com/bcem_uniandes","_blank")}},{id:"lang-pt-br",title:"pt-br",section:"Languages",handler:()=>{window.location.href="/pt-br/publications/"}},{id:"light-theme",title:"Change theme to light",description:"Change the theme of the site to Light",section:"Theme",handler:()=>{setThemeSetting("light")}},{id:"dark-theme",title:"Change theme to dark",description:"Change the theme of the site to Dark",section:"Theme",handler:()=>{setThemeSetting("dark")}},{id:"system-theme",title:"Use system default theme",description:"Change the theme of the site to System Default",section:"Theme",handler:()=>{setThemeSetting("system")}}];</script> <script src="/assets/js/shortcut-key.js"></script> </body> </html>